From 52889453c13258b3fd036bbb7a8b37df132505e3 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 8 Feb 2011 17:59:48 +0100
Subject: [PATCH] dracut-functions: write to $HOME/dracut.log instead of /tmp/dracut.log

---
 dracut-functions |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index e80a638..9ac0bba 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -65,8 +65,7 @@ is_func() {
 if ! [[ $dracutlogfile ]]; then
     [[ $dracutbasedir = /usr/share/dracut ]] && \
         dracutlogfile=/var/log/dracut.log || \
-        dracutlogfile=/tmp/dracut.log
-#    [[ -w $dracutlogfile ]] || dracutlogfile=/tmp/dracut.log
+        dracutlogfile=$HOME/dracut.log
     if [[ -w $dracutlogfile ]]; then
         >"$dracutlogfile"
     fi
