From d693f34afb5f4fccb75e44da2fa8378a67a0e68e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 13 Apr 2010 12:54:12 +0200
Subject: [PATCH 03/15] dmraid: parse different error messages

---
 modules.d/90dmraid/dmraid.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90dmraid/dmraid.sh b/modules.d/90dmraid/dmraid.sh
index 4e47e2d..dba655b 100755
--- a/modules.d/90dmraid/dmraid.sh
+++ b/modules.d/90dmraid/dmraid.sh
@@ -10,7 +10,7 @@ DM_CLEANUP="no"
 info "Scanning for dmraid devices $DM_RAIDS"
 SETS=$(dmraid -c -s)
 
-if [ "$SETS" = "no raid disks" ]; then
+if [ "$SETS" = "no raid disks" -o "$SETS" = "no raid sets" ]; then
     return
 fi
 
-- 
1.7.0.1

