From 59a083d8d18e3c9364c62740e2307679c2d6c7d2 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 18 Jun 2010 12:52:52 +0200
Subject: [PATCH 4/8] fips: fixes copy&paste error for "check"

---
 modules.d/01fips/check |   16 +---------------
 1 files changed, 1 insertions(+), 15 deletions(-)

diff --git a/modules.d/01fips/check b/modules.d/01fips/check
index e5cd87e..9d8d2e6 100755
--- a/modules.d/01fips/check
+++ b/modules.d/01fips/check
@@ -1,19 +1,5 @@
 #!/bin/bash
 
-. $dracutfunctions
-
-for program in ip arping; do 
-  which $program >/dev/null 2>&1
-  if [ $? -ne 0 ]; then
-    dwarning "Could not find program \"$program\" required by network." 
-    exit 1
-  fi
-done
-for program in dhclient brctl; do
-  which $program >/dev/null 2>&1
-  if [ $? -ne 0 ]; then
-    dwarning "Could not find program \"$program\" it might be required by network." 
-  fi
-done
+[[ $1 = -d ]] && exit 0
 exit 255
 
-- 
1.7.0.1

