From 39fe00ef600879279f01289007433bd7f9b84dd9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 22 Aug 2013 09:39:38 +0200
Subject: [PATCH] dracut.asc: document debugging dracut on shutdown

---
 dracut.asc | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/dracut.asc b/dracut.asc
index e0431dc..c011658 100644
--- a/dracut.asc
+++ b/dracut.asc
@@ -744,6 +744,23 @@ the dracut shell
 ==== Additional dracut boot parameters
 For more debugging options, see <<dracutkerneldebug>> in <<dracutcmdline7>>.
 
+
+[[debugging-dracut-on-shutdown]]
+==== Debugging dracut on shutdown
+
+To debug the shutdown sequence on systemd systems, you can _rd.break_
+on _pre-shutdown_ or _shutdown_.
+
+To do this from an already booted system:
+----
+# mkdir -p /run/initramfs/etc/cmdline.d
+# echo "rd.break=pre-shutdown" > /run/initramfs/etc/cmdline.d/debug.conf
+# touch /run/initramfs/.need_shutdown
+----
+
+This will give you a dracut shell after the system pivot'ed back in the initramfs.
+
+
 = Developer Manual
 
 == dracut Components
