From 429aa1a2c29dd42fbc91e956a8a77675773cc46d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 3 Sep 2009 17:11:03 +0200
Subject: [PATCH 5/6] mdraid: add grep for convenience

---
 modules.d/90mdraid/install |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90mdraid/install b/modules.d/90mdraid/install
index 60f289a..497e5b0 100755
--- a/modules.d/90mdraid/install
+++ b/modules.d/90mdraid/install
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-dracut_install mdadm partx
+dracut_install mdadm partx grep
 
 
 # XXX: mdmon really needs to run as non-root?
-- 
1.6.2.5

