From a16ebfd56abfb02f28b72a74c68c24501e2a75d7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 29 Mar 2011 11:30:27 +0200
Subject: [PATCH] dracut-functions: bashified ret=ret+$? to ret+=$?

---
 dracut-functions |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 2accaec..2fc2471 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -787,7 +787,7 @@ instmods() {
                 # it may require
                 for_each_kmod_dep install_kmod_with_fw $mod \
                     --set-version $kernel ${moddirname}
-                ret=$((ret+$?))
+                ((ret+=$?))
                 ;;
         esac
         shift
