From 5297d4f5b2167d26fee368d8014bd72ed93538a4 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Jul 2012 15:36:58 +0200
Subject: [PATCH] lvm/lvm_scan.sh: udevadm settle after lvm scan

---
 modules.d/90lvm/lvm_scan.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/90lvm/lvm_scan.sh b/modules.d/90lvm/lvm_scan.sh
index 17c29bc..d86be9b 100755
--- a/modules.d/90lvm/lvm_scan.sh
+++ b/modules.d/90lvm/lvm_scan.sh
@@ -129,4 +129,6 @@ if [ "$lvmwritten" ]; then
 fi
 unset lvmwritten
 
+udevadm settle
+
 need_shutdown
