From f8e0c7cc6296909cff92f718ed360ff9ef1f7bae Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 5 Mar 2018 09:55:39 +0100
Subject: [PATCH] kernel-modules: add mmc/core for arm

---
 modules.d/90kernel-modules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 94592e6b..7de44735 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -36,6 +36,7 @@ installkernel() {
                 "=drivers/hwspinlock" \
                 "=drivers/i2c/busses" \
                 "=drivers/mfd" \
+                "=drivers/mmc/core" \
                 "=drivers/phy" \
                 "=drivers/power" \
                 "=drivers/regulator" \

