From 4456861f7edc8861a32bdd8ca3d3e49f602837e8 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 20 Jul 2011 19:06:08 +0200
Subject: [PATCH] 90dm/dm-shutdown.sh: put shutdown in function

---
 modules.d/90dm/dm-shutdown.sh |   17 +++++++++++++++--
 1 files changed, 15 insertions(+), 2 deletions(-)

diff --git a/modules.d/90dm/dm-shutdown.sh b/modules.d/90dm/dm-shutdown.sh
index 026eb26..3e668f8 100644
--- a/modules.d/90dm/dm-shutdown.sh
+++ b/modules.d/90dm/dm-shutdown.sh
@@ -1,2 +1,15 @@
-echo "Disassembling device-mapper devices"
-dmsetup -v remove_all
+#!/bin/sh
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+_do_dm_shutdown() {
+    local ret
+    info "Disassembling device-mapper devices"
+    dmsetup -v remove_all
+    ret=$?
+#info "dmsetup ls --tree"
+#dmsetup ls --tree 2>&1 | vinfo
+    return $ret
+}
+
+_do_dm_shutdown
+
