From 96f3cf1d27268f3784f420ebb52228f3e6b08e77 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 7 Sep 2009 11:40:20 +0200
Subject: [PATCH 7/8] 95udev-rules: fixed c&p bug which did not install 61-persistent-storage.rules

---
 modules.d/95udev-rules/install |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/95udev-rules/install b/modules.d/95udev-rules/install
index cf22d1e..695e472 100755
--- a/modules.d/95udev-rules/install
+++ b/modules.d/95udev-rules/install
@@ -23,7 +23,7 @@ if [ ! -x /lib/udev/vol_id ]; then
 else
     inst_rules "$moddir/59-persistent-storage-volid.rules"
 fi
-inst_rules "$moddir/61-persistent-storage-volid.rules"
+inst_rules "$moddir/61-persistent-storage.rules"
     
 for i in \
 ata_id \
-- 
1.6.2.5

