From ea216a650ef738e324253f27b59d74e3d8aa6062 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 29 Mar 2011 14:45:08 +0200
Subject: [PATCH] dracut: removed extra char in "Including modules done"

---
 dracut |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dracut b/dracut
index c156cdb..404b9db 100755
--- a/dracut
+++ b/dracut
@@ -493,7 +493,7 @@ for moddir in "$dracutbasedir/modules.d"/[0-9][0-9]*; do
     fi
 done
 unset moddir
-dinfo "*** Including modules' done ***"
+dinfo "*** Including modules done ***"
 
 ## final stuff that has to happen
 
