From cef379bb0391356ecb9dd01a30e7af5bc8f19655 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 24 Jul 2012 11:52:28 +0200
Subject: [PATCH] i18n/10-console.rules: move console_init to the initqueue

---
 modules.d/10i18n/10-console.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/10i18n/10-console.rules b/modules.d/10i18n/10-console.rules
index 80ed090..385dff0 100644
--- a/modules.d/10i18n/10-console.rules
+++ b/modules.d/10i18n/10-console.rules
@@ -1,2 +1,2 @@
 # Console initialization - keyboard, font, etc.
-KERNEL=="tty0",		RUN+="/lib/udev/console_init $root/$name"
+KERNEL=="tty0",		RUN+="/sbin/initqueue --unique --name console_init_$name /lib/udev/console_init $root/$name"
