From 8d28614e4f6d270e57371cde4ea9f1fe776cdb35 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Sun, 10 Mar 2013 14:44:16 +0100
Subject: [PATCH] 51-dracut-rescue.install: do not unset installonly on remove

---
 51-dracut-rescue.install | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/51-dracut-rescue.install b/51-dracut-rescue.install
index 844e578..e5781d3 100755
--- a/51-dracut-rescue.install
+++ b/51-dracut-rescue.install
@@ -55,12 +55,6 @@ case "$COMMAND" in
         [[ -f $LOADER_ENTRY ]] || exit 0
 
         rm -f "$LOADER_ENTRY" "$3"/initrd-rescue
-
-        if command -v yumdb &>/dev/null; then
-            if [[ $(yumdb get installonly kernel-$KERNEL_VERSION 2>/dev/null) == *installonly\ \=\ keep* ]]; then
-                yumdb del installonly kernel-$KERNEL_VERSION >/dev/null
-            fi
-        fi
         ;;
 
     *)
