From 00d89f51701a89543c67715beedc70b903244f08 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 20 Jul 2011 21:10:58 +0200
Subject: [PATCH] TEST-10-RAID/test-init: turn off debug for shutdown

---
 test/TEST-10-RAID/test-init |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/test/TEST-10-RAID/test-init b/test/TEST-10-RAID/test-init
index 029084a..4c8c9e1 100755
--- a/test/TEST-10-RAID/test-init
+++ b/test/TEST-10-RAID/test-init
@@ -14,6 +14,7 @@ echo "made it to the rootfs!"
 strstr "$CMDLINE" "rd.shell" && sh -i
 echo "Powering down."
 mount -n -o remount,ro /
+echo " rd.debug=0 " > /run/initramfs/etc/cmdline
 if [ -e /lib/systemd/systemd-shutdown ]; then
 	exec /lib/systemd/systemd-shutdown poweroff
 fi
