From 4f4a79b1a13f9961030a3d97743274116f72054a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 7 Sep 2009 17:10:19 +0200
Subject: [PATCH 10/12] add scp to debug module

---
 modules.d/95debug/install |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/95debug/install b/modules.d/95debug/install
index 1c7afaf..cd8c48c 100755
--- a/modules.d/95debug/install
+++ b/modules.d/95debug/install
@@ -1,3 +1,3 @@
 #!/bin/bash
 dracut_install -o ps grep more cat rm strace free showmount 
-dracut_install -o ping netstat rpcinfo vi
+dracut_install -o ping netstat rpcinfo vi scp
-- 
1.6.2.5

