From f2271428d4760c7026dda9093c14fb1bf9035691 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 24 Jul 2012 14:08:32 +0200
Subject: [PATCH] dracut-functions.sh: small error print correction

---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 6358078..7f834f0 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -1135,7 +1135,7 @@ for_each_module_dir() {
     for _mod in $_modcheck; do
         strstr "$mods_to_load" "$_mod" && continue
         strstr "$omit_dracutmodules" "$_mod" && continue
-        derror "Dracut module \"$_mod\" cannot be found or installed."
+        derror "Dracut module '$_mod' cannot be found or installed."
     done
 }
 
