From d2a9c4a8e7920cc01107b7cbab953da5c6cab100 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Jun 2012 10:41:59 +0200
Subject: [PATCH] dracut.sh: mkdir of $libdirs at then end

---
 dracut.sh |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/dracut.sh b/dracut.sh
index 78e7ac8..a1b39e3 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -688,13 +688,14 @@ export initdir dracutbasedir dracutmodules drivers \
 [[ $prefix ]] && ln -sfn "${prefix#/}/lib" "$initdir/lib"
 
 if [[ $prefix ]]; then
-    for d in bin etc lib $libdirs sbin tmp usr var; do
+    for d in bin etc lib sbin tmp usr var $libdirs; do
+        strstr "$d" "/" && continue
         ln -sfn "${prefix#/}/${d#/}" "$initdir/$d"
     done
 fi
 
 if [[ $kernel_only != yes ]]; then
-    for d in usr/bin usr/sbin bin etc lib $libdirs sbin tmp usr var var/log var/run var/lock; do
+    for d in usr/bin usr/sbin bin etc lib sbin tmp usr var var/log var/run var/lock $libdirs; do
         [[ -e "${initdir}${prefix}/$d" ]] && continue
         if [ -L "/$d" ]; then
             inst_symlink "/$d" "${prefix}/$d"
