From dd5875499ece9dbc90e10eafd0073ee15d0c86a4 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 11 Mar 2013 16:31:48 +0100
Subject: [PATCH] dracut.sh: add swap partitions to host-only setup

---
 dracut.sh | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 8306733..bb49299 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -794,6 +794,23 @@ if [[ $hostonly ]]; then
         mountpoint "$mp" >/dev/null 2>&1 || continue
         push host_devs $(readlink -f "/dev/block/$(find_block_device "$mp")")
     done
+
+    while read dev type rest; do
+        [[ -b $dev ]] || continue
+        [[ "$type" == "partition" ]] || continue
+        while read _d _m _t _o _r; do
+            [[ "$_d" == \#* ]] && continue
+            [[ $_d ]] || continue
+            [[ $_t != "swap" ]] || [[ $_m != "swap" ]] && continue
+            [[ "$_o" == *noauto* ]] && continue
+            [[ "$_d" == UUID\=* ]] && _d="/dev/disk/by-uuid/${_d#UUID=}"
+            [[ "$_d" == LABEL\=* ]] && _d="/dev/disk/by-label/$_d#LABEL=}"
+            [[ "$_d" -ef "$dev" ]] || continue
+            push host_devs $(readlink -f $dev)
+            break
+        done < /etc/fstab
+    done < /proc/swaps
+
 fi
 
 _get_fs_type() (
