From e9e93e2ec6f58cfcd7e4680827b242a90a038710 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 15 Apr 2010 12:47:52 +0200
Subject: [PATCH 13/15] kernel-modules: only remove ocfs2, if all filesystems are installed

---
 modules.d/90kernel-modules/installkernel |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/modules.d/90kernel-modules/installkernel b/modules.d/90kernel-modules/installkernel
index 4848f2a..90b535e 100755
--- a/modules.d/90kernel-modules/installkernel
+++ b/modules.d/90kernel-modules/installkernel
@@ -17,14 +17,15 @@ if [[ -z $drivers ]]; then
     if ! [[ $hostonly ]]; then
 	if [[ -z $filesystems ]]; then
 	    instmods '=fs'
+	    # hardcoded list of exceptions
+	    # to save a lot of space
+	    rm -fr ${initdir}/lib/modules/*/kernel/fs/ocfs2
 	else
 	    instmods $filesystems
 	fi
     else
 	hostonly='' instmods $(get_fs_type "/dev/block/$(find_root_block_device)")
     fi
-    # hardcoded list of exceptions
-    rm -fr ${initdir}/lib/modules/*/kernel/fs/ocfs2
 else
   hostonly='' instmods $drivers $filesystems
 fi
-- 
1.7.0.1

