From 3c3b2cf093302b1441206cc690bd77c487e0b06d Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Thu, 22 Mar 2018 16:25:46 +0100
Subject: [PATCH] spec: add missing obsoletes and provides for removed fips
 subpackages

---
 dracut.spec | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index 5a31a1f3..3bfbd5c4 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -60,6 +60,11 @@ Obsoletes: mkinitrd < 2.6.1
 Provides: mkinitrd = 2.6.1
 %endif
 
+Obsoletes: dracut-fips <= 047
+Provides:  dracut-fips = %{version}-%{release}
+Obsoletes: dracut-fips-aesni <= 047
+Provides:  dracut-fips-aesni = %{version}-%{release}
+
 Requires: bash >= 4
 Requires: coreutils
 Requires: cpio

