From 87bd8907a64991445af552740806d3949fc6d6ca Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 26 Aug 2013 10:19:49 +0200
Subject: [PATCH] i18n: check for setfont and loadkeys binaries

---
 modules.d/10i18n/module-setup.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules.d/10i18n/module-setup.sh b/modules.d/10i18n/module-setup.sh
index be9d90b..2a83728 100755
--- a/modules.d/10i18n/module-setup.sh
+++ b/modules.d/10i18n/module-setup.sh
@@ -4,6 +4,11 @@
 
 check() {
     [[ "$mount_needs" ]] && return 1
+
+    for i in setfont loadkeys kbd_mode; do
+        type -P "$i" >/dev/null || return 1
+    done
+
     return 0
 }
 
