From fede5bbcdad4b35baba06eb5c5aebc0df4aa0347 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 15 Apr 2010 14:38:38 +0200
Subject: [PATCH 14/15] dracut.spec: add btrfs module

---
 dracut.spec |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index c4bc00d..e2b6f44 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -180,6 +180,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_datadir}/dracut/modules.d/10rpmversion
 %{_datadir}/dracut/modules.d/50plymouth
 %{_datadir}/dracut/modules.d/60xen
+%{_datadir}/dracut/modules.d/90btrfs
 %{_datadir}/dracut/modules.d/90crypt
 %{_datadir}/dracut/modules.d/90dm
 %{_datadir}/dracut/modules.d/90dmraid
-- 
1.7.0.1

