From e1e4ee9d7c005f61b6ad88d92ef8993d2cfe92be Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 20 Nov 2014 16:22:51 +0100
Subject: [PATCH] TODO: update

---
 TODO | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TODO b/TODO
index 78dae0f..917b97c 100644
--- a/TODO
+++ b/TODO
@@ -28,6 +28,7 @@ GENERATOR TODO
 
 - report errors on missing files in check()
 - remove wait for swap devs, if no "resume=" is given on the kernel command line
+- remove wait for swap devs, if the "resume" dracut module is not included (omitted)
 - add presets (predefined set of modules)
 - add interpreter/plugin-scripts to be sourced at the beginning or end (can use dracut-functions)
 - add mechanism for module specific command line options
