From 33782aebbffa005415e1a03deeae837d8cf5665e Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 30 Jul 2018 02:26:01 +0900
Subject: [PATCH] spec: warpclock is not available on s390 or s390x

---
 dracut.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index 9b9d497a..ce69fa1f 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -224,6 +224,7 @@ rm -fr -- $RPM_BUILD_ROOT/%{dracutlibdir}/modules.d/98integrity
 
 %ifnarch s390 s390x
 # remove architecture specific modules
+rm -fr -- $RPM_BUILD_ROOT/%{dracutlibdir}/modules.d/00warpclock
 rm -fr -- $RPM_BUILD_ROOT/%{dracutlibdir}/modules.d/80cms
 rm -fr -- $RPM_BUILD_ROOT/%{dracutlibdir}/modules.d/81cio_ignore
 rm -fr -- $RPM_BUILD_ROOT/%{dracutlibdir}/modules.d/91zipl
@@ -316,7 +317,9 @@ install -m 0755 51-dracut-rescue-postinst.sh $RPM_BUILD_ROOT%{_sysconfdir}/kerne
 %endif
 %{dracutlibdir}/modules.d/00bash
 %{dracutlibdir}/modules.d/00systemd
+%ifnarch s390 s390x
 %{dracutlibdir}/modules.d/00warpclock
+%endif
 %{dracutlibdir}/modules.d/01systemd-initrd
 %{dracutlibdir}/modules.d/03modsign
 %{dracutlibdir}/modules.d/03rescue

