From 8570b544e96d150263c639b9411270488a95ad2b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 20 Jul 2011 21:40:50 +0200
Subject: [PATCH] TEST-10-RAID/test-init: add comment for rd.break=shutdown

---
 test/TEST-10-RAID/test-init |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/test/TEST-10-RAID/test-init b/test/TEST-10-RAID/test-init
index 4c8c9e1..02c0d1d 100755
--- a/test/TEST-10-RAID/test-init
+++ b/test/TEST-10-RAID/test-init
@@ -14,7 +14,8 @@ echo "made it to the rootfs!"
 strstr "$CMDLINE" "rd.shell" && sh -i
 echo "Powering down."
 mount -n -o remount,ro /
-echo " rd.debug=0 " > /run/initramfs/etc/cmdline
+#echo " rd.break=shutdown " >> /run/initramfs/etc/cmdline
+echo " rd.debug=0 " >> /run/initramfs/etc/cmdline
 if [ -e /lib/systemd/systemd-shutdown ]; then
 	exec /lib/systemd/systemd-shutdown poweroff
 fi
