From 8652d5493d139105620ab82f34334f8acae13f13 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 6 Aug 2013 11:01:45 +0200
Subject: [PATCH] 51-dracut-rescue.install: fixed rescue image creation

---
 51-dracut-rescue.install | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/51-dracut-rescue.install b/51-dracut-rescue.install
index 39d01a7..b655bc2 100755
--- a/51-dracut-rescue.install
+++ b/51-dracut-rescue.install
@@ -6,6 +6,9 @@ export LANG=C
 
 COMMAND="$1"
 KERNEL_VERSION="$2"
+BOOT_DIR_ABS="${3%/*}/0-rescue"
+KERNEL_IMAGE="$4"
+
 
 dropindirs_sort()
 {
@@ -50,7 +53,6 @@ fi
 
 LOADER_ENTRY="/boot/loader/entries/${MACHINE_ID}-0-rescue.conf"
 BOOT_DIR="/${MACHINE_ID}/0-rescue"
-BOOT_DIR_ABS="/boot${BOOT_DIR}"
 
 ret=0
 
