From da119ab18c4813b543f16303a62a6b718c6abcb3 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 26 Aug 2013 12:46:51 +0200
Subject: [PATCH] dmraid/parse-dm.sh: if rd.md=0, use dmraid for imsm and ddf

---
 modules.d/90dmraid/parse-dm.sh | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/modules.d/90dmraid/parse-dm.sh b/modules.d/90dmraid/parse-dm.sh
index 76dfb5d..6d21d4f 100755
--- a/modules.d/90dmraid/parse-dm.sh
+++ b/modules.d/90dmraid/parse-dm.sh
@@ -7,12 +7,16 @@ if ! getargbool 1 rd.dm -d -n rd_NO_DM || getarg "rd.dm=0" -d nodmraid; then
     udevproperty rd_NO_DM=1
 fi
 
-if  ! command -v mdadm >/dev/null || ! getargbool 1 rd.md.imsm -d -n rd_NO_MDIMSM -n noiswmd; then
+if  ! command -v mdadm >/dev/null \
+    || ! getargbool 1 rd.md.imsm -d -n rd_NO_MDIMSM -n noiswmd \
+    || ! getargbool 1 rd.md -d -n rd_NO_MD; then
     info "rd.md.imsm=0: no MD RAID for imsm/isw raids"
     udevproperty rd_NO_MDIMSM=1
 fi
 
-if  ! command -v mdadm >/dev/null || ! getargbool 1 rd.md.ddf -n rd_NO_MDDDF -n noddfmd; then
+if  ! command -v mdadm >/dev/null \
+    || ! getargbool 1 rd.md.ddf -n rd_NO_MDDDF -n noddfmd \
+    || ! getargbool 1 rd.md -d -n rd_NO_MD; then
     info "rd.md.ddf=0: no MD RAID for SNIA ddf raids"
     udevproperty rd_NO_MDDDF=1
 fi
