From bebb12fd0ec3cbb88006e384a9a277391a2f3a37 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 11 Jun 2012 17:11:49 +0200
Subject: [PATCH] network/ifup.sh: default to dhcp for BOOTIF

---
 modules.d/40network/ifup.sh |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/modules.d/40network/ifup.sh b/modules.d/40network/ifup.sh
index f785968..35ae3c6 100755
--- a/modules.d/40network/ifup.sh
+++ b/modules.d/40network/ifup.sh
@@ -213,6 +213,13 @@ if [ "$netif" = "$vlanname" ] && [ ! -e /tmp/net.$vlanname.up ]; then
     ip link add dev "$vlanname" link "$phydevice" type vlan id "$(get_vid $vlanname; echo $?)"
 fi
 
+# BOOTIF= defaults to dhcpv4
+bootif=$(getargs BOOTIF=)
+if [ -n "$bootif" ] ; then
+    do_dhcp -4
+    exit 0
+fi
+
 # Specific configuration, spin through the kernel command line
 # looking for ip= lines
 for p in $(getargs ip=); do
