From 8552a3270caeec41f4610f0081c3d550cb4054eb Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 27 Aug 2013 11:34:39 +0200
Subject: [PATCH] dracut-functions.sh:get_devpath_block() make local vars local

---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 9b24419..4c62d6b 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -300,7 +300,7 @@ get_maj_min() {
 # get_devpath_block <device>
 # get the DEVPATH in /sys of a block device
 get_devpath_block() {
-    local majmin i
+    local _majmin _i
     _majmin=$(get_maj_min "$1")
 
     for _i in /sys/block/*/dev /sys/block/*/*/dev; do
