From 43fed15129dc4916c37ebd8980aba78b6609943b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 12 Jun 2012 11:35:49 -0400
Subject: [PATCH] dracut.sh: mkdir $initdir/lib/dracut

---
 dracut.sh |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index a1b39e3..c26e0f8 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -731,7 +731,10 @@ if [[ $kernel_only != yes ]]; then
         mkdir -m 0755 -p ${initdir}/lib/dracut/hooks/$_d
     done
     if [[ "$UID" = "0" ]]; then
-        cp -a /dev/kmsg /dev/null /dev/console $initdir/dev
+        for i in /dev/kmsg /dev/null /dev/console; do
+            [ -e $i ] || continue
+            cp -a $i $initdir/dev
+        done
     fi
 fi
 
@@ -780,6 +783,7 @@ done
 unset moddir
 
 for i in $modules_loaded; do
+    mkdir -p $initdir/lib/dracut
     echo "$i" >> $initdir/lib/dracut/modules.txt
 done
 
