From 0a7d7f63106ce5a7f9869aacb21d0474621811e7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Jan 2016 09:17:40 +0100
Subject: [PATCH] dracut.cmdline: remove extra ":" from ip= doc

---
 dracut.cmdline.7.asc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.cmdline.7.asc b/dracut.cmdline.7.asc
index e160e3b..c4c5588 100644
--- a/dracut.cmdline.7.asc
+++ b/dracut.cmdline.7.asc
@@ -492,7 +492,7 @@ cannot be used in conjunction with the **ifname** argument for the
 same <interface>.
 =====================
 
-**ip=**__<client-IP>__:[__<peer>__]:__<gateway-IP>__:__<netmask>__:__<client_hostname>__:__<interface>__:__{none|off|dhcp|on|any|dhcp6|auto6|ibft}__:[:[__<mtu>__][:__<macaddr>__]]::
+**ip=**__<client-IP>__:[__<peer>__]:__<gateway-IP>__:__<netmask>__:__<client_hostname>__:__<interface>__:__{none|off|dhcp|on|any|dhcp6|auto6|ibft}__[:[__<mtu>__][:__<macaddr>__]]::
     explicit network configuration. If you want do define a IPv6 address, put it
     in brackets (e.g. [2001:DB8::1]). This parameter can be specified multiple
     times. __<peer>__ is optional and is the address of the remote endpoint
