From a762dedb60aef8f43a625e1e3c10c97664f1d74a Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Thu, 9 Aug 2018 12:14:20 +0200
Subject: [PATCH] dracut-functions: fix the word splitting

---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 7c408f83..1431dd18 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -683,7 +683,7 @@ get_loaded_kernel_modules ()
     local modules=( )
     while read _module _size _used _used_by; do
         modules+=( "$_module" )
-    done <<< $(lsmod | sed -n '1!p')
+    done <<< "$(lsmod | sed -n '1!p')"
     printf '%s\n' "${modules[@]}" | sort
 }
 

