From 82eea29797588621fc10f7f2e335ec5bf727164f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 22 Jul 2011 14:17:59 +0200
Subject: [PATCH] 99base/dracut-lib.sh: don't be quiet on rd.debug

---
 modules.d/99base/dracut-lib.sh |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index 6807b53..f65e853 100755
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -272,6 +272,7 @@ check_quiet() {
     if [ -z "$DRACUT_QUIET" ]; then
         DRACUT_QUIET="yes"
         getargbool 0 rd.info -y rdinfo && DRACUT_QUIET="no"
+        getargbool 0 rd.debug -y rdinitdebug && DRACUT_QUIET="no"
         getarg quiet || DRACUT_QUIET="yes"
     fi
 }
