From 99801b6e39b19bd6c4dfe3c033cf2cfb8a38ccd1 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 7 May 2014 14:54:56 +0200
Subject: [PATCH] Do not log to kmsg/syslog and files for "--print-cmdline"

---
 dracut.sh | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index c1ef4745c6..c4163bcfec 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -834,6 +834,13 @@ trap 'exit 1;' SIGINT
 export DRACUT_KERNEL_LAZY="1"
 export DRACUT_RESOLVE_LAZY="1"
 
+if [[ $print_cmdline ]]; then
+    stdloglvl=0
+    sysloglvl=0
+    fileloglvl=0
+    kmsgloglvl=0
+fi
+
 if [[ -f $dracutbasedir/dracut-functions.sh ]]; then
     . $dracutbasedir/dracut-functions.sh
 else
@@ -907,6 +914,7 @@ for ((i=0; i < ${#dracut_args[@]}; i++)); do
         dracut_args[$i]="\"${dracut_args[$i]}\""
         #" keep vim happy
 done
+
 dinfo "Executing: $0 ${dracut_args[@]}"
 
 [[ $do_list = yes ]] && {
-- 
1.9.3

