From b8060ef781a980afebec4412666a5b41c5cada2c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Mar 2013 11:07:36 +0100
Subject: [PATCH] systemd: add sysctl.d and sysctl.conf

---
 modules.d/98systemd/module-setup.sh | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index 6b00be5..bc4f63c 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -108,7 +108,8 @@ install() {
         journalctl systemctl echo swapoff systemd-cgls
 
     dracut_install -o \
-        /usr/lib/modules-load.d/*.conf
+        /usr/lib/modules-load.d/*.conf \
+        /usr/lib/sysctl.d/*.conf
 
     modules_load_get() {
         local _line i
@@ -137,10 +138,11 @@ install() {
             /etc/hostname \
             /etc/machine-id \
             /etc/vconsole.conf \
-            /etc/locale.conf
+            /etc/locale.conf \
+            /etc/modules-load.d/*.conf \
+            /etc/sysctl.d/*.conf \
+            /etc/sysctl.conf
 
-        dracut_install -o \
-            /etc/modules-load.d/*.conf
         _mods=$(modules_load_get /etc/modules-load.d)
         [[ $_mods ]] && instmods $_mods
     else
