From 985377435120ce2b2ba48c9236f3f788a5db822a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 31 Mar 2011 14:10:22 +0200
Subject: [PATCH] base/init: reset PATH after the /run move

---
 modules.d/99base/init |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 83d9568..0f18b44 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -373,7 +373,6 @@ info "Switching root"
 
 wait_for_loginit
 
-export PATH="$OLD_PATH"
 
 if [ -d "$NEWROOT"/run ]; then
     mount --move /run "$NEWROOT"/run
@@ -388,6 +387,8 @@ else
     umount -l /run
 fi
 
+export PATH="$OLD_PATH"
+
 if [ -f /etc/capsdrop ]; then
     . /etc/capsdrop
     info "Calling $INIT with capabilities $CAPS_INIT_DROP dropped."
