From a844fb1552d7867c77caa027ea38249c26866479 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 25 Jul 2012 14:53:23 +0200
Subject: [PATCH] dracut.asc: add doc about journalctl

---
 dracut.asc | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/dracut.asc b/dracut.asc
index ca896ee..1791e40 100644
--- a/dracut.asc
+++ b/dracut.asc
@@ -459,6 +459,13 @@ dracut shell commands are printed as they are executed
 # less /run/initramfs/init.log
 # dmesg | less
 ----
+. With dracut >= 022 and systemd, you can inspect the rd.debug output with:
+----
+# journalctl -ab
+----
+
+If you want to save that output, simply mount /boot by hand or insert an USB stick and mount that.
+Then you can store the output for later inspection.
 
 [[information-to-include-in-your-report]]
 === Information to include in your report
@@ -472,7 +479,7 @@ configuration file (e.g. _/etc/grub.conf_) or from _/proc/cmdline_.
 * A copy of your disk partition information from _/etc/fstab_, which might be
 obtained booting an old working initramfs or a rescue medium.
 * A device listing from device-mapper. This can be obtained by running the
-command 
+command
 +
 ----
 # dmsetup ls --tree
