From 3659d64df3fe87e2ec2ca6a55c8f456dabb953db Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 28 Nov 2014 15:32:17 +0100
Subject: [PATCH] fips: add libfreeblpriv3.so and libfreeblpriv3.chk

---
 modules.d/01fips/module-setup.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index aab96da..01e84e5 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -40,7 +40,8 @@ install() {
 
     inst_libdir_file libsoftokn3.so libsoftokn3.so \
         libsoftokn3.chk libfreebl3.so libfreebl3.chk \
-        libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10
+        libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10 \
+        libfreeblpriv3.so and libfreeblpriv3.chk
 
     inst_multiple -o prelink
     inst_simple /etc/system-fips
