From e094f77d93e2e68aeb3d4488c3f5990ddd47081b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 May 2014 13:53:38 +0200
Subject: [PATCH] ifcfg/write-ifcfg.sh: "IPV6INIT=yes" check also for non-dhcp

---
 modules.d/45ifcfg/write-ifcfg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index 5dc351d584..457a9ac4ac 100755
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -120,9 +120,9 @@ for netup in /tmp/net.*.did-setup ; do
         echo "ONBOOT=yes"
         echo "NETBOOT=yes"
         echo "UUID=\"$uuid\""
+        strstr "$(ip -6 addr show dev $netif)" 'inet6' && echo "IPV6INIT=yes"
         if [ -f /tmp/dhclient.$netif.lease ]; then
             [ -f /tmp/dhclient.$netif.dhcpopts ] && . /tmp/dhclient.$netif.dhcpopts
-            strstr "$(ip -6 addr show dev $netif)" 'inet6' && echo "IPV6INIT=yes"
             if [ -f /tmp/net.$netif.has_ibft_config ]; then
                 echo "BOOTPROTO=ibft"
             else
-- 
1.9.3

