From 2c9bcac1136c6a18aedcdcdac5789ccf5cd18492 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 10 Oct 2017 13:39:17 +0200
Subject: [PATCH] dracut-install: char* -> const char*

---
 install/dracut-install.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install/dracut-install.c b/install/dracut-install.c
index 95425e8d..82e882cc 100644
--- a/install/dracut-install.c
+++ b/install/dracut-install.c
@@ -1392,7 +1392,7 @@ static int install_modules(int argc, char **argv)
         struct kmod_module *mod = NULL, *mod_o = NULL;
 
         const char *modname = NULL;
-        char *abskpath = NULL;
+        const char *abskpath = NULL;
         char *p;
         int i;
 

