From 7d17d6c976d705deebf3da46fd546b1fe7e1dd65 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Mar 2013 10:51:17 +0100
Subject: [PATCH] plymouth: do not install hooks in systemd mode

---
 modules.d/50plymouth/module-setup.sh | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/modules.d/50plymouth/module-setup.sh b/modules.d/50plymouth/module-setup.sh
index 1c5dad5..f754e24 100755
--- a/modules.d/50plymouth/module-setup.sh
+++ b/modules.d/50plymouth/module-setup.sh
@@ -17,12 +17,16 @@ install() {
         . "$moddir"/plymouth-populate-initrd.sh
     else
         PLYMOUTH_POPULATE_SOURCE_FUNCTIONS="$dracutfunctions" \
-            /usr/libexec/plymouth/plymouth-populate-initrd -t $initdir
+            /usr/libexec/plymouth/plymouth-populate-initrd -t "$initdir"
     fi
 
-    inst_hook pre-pivot 90 "$moddir"/plymouth-newroot.sh
-    inst_hook pre-trigger 10 "$moddir"/plymouth-pretrigger.sh
     inst_hook emergency 50 "$moddir"/plymouth-emergency.sh
+
     dracut_install readlink
+
+    if ! dracut_module_included "systemd"; then
+        inst_hook pre-trigger 10 "$moddir"/plymouth-pretrigger.sh
+        inst_hook pre-pivot 90 "$moddir"/plymouth-newroot.sh
+    fi
 }
 
