From d93fe0fa35d29762a21a44a9644ba37e352e8b7a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 9 Aug 2013 15:35:20 +0200
Subject: [PATCH] lsinitrd.sh: fixed case

---
 lsinitrd.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lsinitrd.sh b/lsinitrd.sh
index ef1c262..faad99a 100755
--- a/lsinitrd.sh
+++ b/lsinitrd.sh
@@ -109,8 +109,7 @@ case $bin in
         CAT="zcat";;
     BZh*)
         CAT="bzcat";;
-    $'\x71\xc7'*)
-    070701)
+    $'\x71\xc7'*|070701)
         CAT="cat";;
     *)
         CAT="xzcat";
