From e045e1f386b2e44e9be88ea53f45090dfc7e095d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 25 Jul 2012 15:20:22 +0200
Subject: [PATCH] Makefile: honor CFLAGS

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 548952d..a109ef0 100644
--- a/Makefile
+++ b/Makefile
@@ -8,6 +8,7 @@ pkglibdir ?= ${libdir}/dracut
 sysconfdir ?= ${prefix}/etc
 bindir ?= ${prefix}/bin
 mandir ?= ${prefix}/share/man
+CFLAGS ?= -std=gnu99 -O2 -g -Wall
 
 man1pages = lsinitrd.1
 
@@ -46,7 +47,7 @@ DRACUT_INSTALL_HEADER = \
         install/util.h
 
 dracut-install: $(DRACUT_INSTALL_SOURCE) $(DRACUT_INSTALL_HEADER)
-	gcc -std=gnu99 -O2 -g -Wall -o dracut-install $(DRACUT_INSTALL_SOURCE)
+	gcc $(CFLAGS) -o dracut-install $(DRACUT_INSTALL_SOURCE)
 
 indent:
 	indent -i8 -nut -br -linux -l120 install/dracut-install.c
