From 11b5db88c2d9290fc438b12a79a50bdd1d327203 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 15 Aug 2014 11:50:31 +0200
Subject: [PATCH] dracut.usage.asc: also add a link for RHEL-7 storage admin
 guide

---
 dracut.usage.asc | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dracut.usage.asc b/dracut.usage.asc
index 50ceb49..6128ec4 100644
--- a/dracut.usage.asc
+++ b/dracut.usage.asc
@@ -259,7 +259,9 @@ local hard drive or it can be served by a PXE/TFTP server.
 
 How to setup your PXE/TFTP server can be found in the
 http://docs.redhat.com/docs/en-US/Red_Hat_Enterprise_Linux/6/html/Storage_Administration_Guide/[Red
-Hat Enterprise Linux Storage Administration Guide].
+Hat Enterprise Linux 6 Storage Administration Guide]
+or http://docs.redhat.com/docs/en-US/Red_Hat_Enterprise_Linux/6/html/Storage_Administration_Guide/[Red
+Hat Enterprise Linux 7 Storage Administration Guide].
 
 If you specify ip=auto on the kernel command line, then dracut asks a dhcp
 server about the ip adress for the machine. The dhcp server can also serve an
