From faabcb23602673f87ec60965b3ff615004371603 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 11 Oct 2017 15:28:20 +0200
Subject: [PATCH] dracut.sh: quote sync file argument

---
 dracut.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/dracut.sh b/dracut.sh
index 995d983e..990700af 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1823,8 +1823,7 @@ fi
 
 command -v restorecon &>/dev/null && restorecon -- "$outfile"
 
-sync $outfile 2> /dev/null
-if [ $? -ne 0 ] ; then
+if ! sync "$outfile" 2> /dev/null; then
     dinfo "dracut: sync operartion on newly created initramfs $outfile failed"
     exit 1
 fi

