From f79e587cf34e9d9c68e87ebad3e5620b0a190581 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 15 Jun 2012 11:11:30 +0200
Subject: [PATCH] dracut.sh: unset some variables

---
 dracut.sh |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 309fd7d..64cd75e 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -217,6 +217,8 @@ push_arg() {
 }
 
 verbosity_mod_l=0
+unset kernel
+unset outfile
 
 while (($# > 0)); do
     case ${1%%=*} in
