From 3e70a25bcf2fb49fbc81d110d3186bddf4f81ad0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 Aug 2013 12:44:10 +0200
Subject: [PATCH] dracut-logger.sh: saveguard logger with --

---
 dracut-logger.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-logger.sh b/dracut-logger.sh
index ff23301..0c505e2 100755
--- a/dracut-logger.sh
+++ b/dracut-logger.sh
@@ -323,7 +323,7 @@ _do_dlog() {
         if [[ "$_dlogfd" ]]; then
             echo "<$(_dlvl2syslvl $lvl)>$msg" >&$_dlogfd
         else
-            logger -t "dracut[$$]" -p $(_lvl2syspri $lvl) "$msg"
+            logger -t "dracut[$$]" -p $(_lvl2syspri $lvl) -- "$msg"
         fi
     fi
 
