From fbdfa57746933d24693ebfcb972f979b34b4401e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Feb 2013 14:17:35 +0100
Subject: [PATCH] systemd/dracut-pre-udev.service: load service on demand

---
 modules.d/98systemd/dracut-pre-udev.service | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/98systemd/dracut-pre-udev.service b/modules.d/98systemd/dracut-pre-udev.service
index 88a8da1..ee2c2e1 100644
--- a/modules.d/98systemd/dracut-pre-udev.service
+++ b/modules.d/98systemd/dracut-pre-udev.service
@@ -15,6 +15,8 @@ Before=systemd-udevd.service dracut-pre-trigger.service
 After=dracut-cmdline.service
 Wants=dracut-cmdline.service
 ConditionPathExists=/etc/initrd-release
+ConditionDirectoryNotEmpty=|/lib/dracut/hooks/pre-udev
+ConditionKernelCommandLine=|rd.break=pre-udev
 
 [Service]
 Type=oneshot
