From dadd2b698ef7d0c069dce58ccad5543337b007c9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 Aug 2013 12:46:03 +0200
Subject: [PATCH] dracut.sh: use local lsinitrd.sh for "-l"

---
 dracut.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 0b674ae..6208c82 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1271,7 +1271,11 @@ mv -- "$outfile.$$" "$outfile"
 dinfo "*** Creating image file done ***"
 
 if (( maxloglvl >= 5 )); then
-    lsinitrd "$outfile"| ddebug
+    if [[ $allowlocal ]]; then
+	"$dracutbasedir/lsinitrd.sh" "$outfile"| ddebug
+    else
+        lsinitrd "$outfile"| ddebug
+    fi
 fi
 
 exit 0
