From cf5229190dfb711f6380afe50917559477711d83 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 13 Jan 2012 11:20:05 +0100
Subject: [PATCH] TEST-10-RAID: fixed TESTDIR handling

---
 test/TEST-10-RAID/test.sh |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/test/TEST-10-RAID/test.sh b/test/TEST-10-RAID/test.sh
index 90758d8..5ac0bc5 100755
--- a/test/TEST-10-RAID/test.sh
+++ b/test/TEST-10-RAID/test.sh
@@ -5,8 +5,8 @@ KVERSION=${KVERSION-$(uname -r)}
 
 # Uncomment this to debug failures
 #DEBUGFAIL="rd.shell"
-DISKIMAGE=$TESTDIR/TEST-10-RAID-root.img
 test_run() {
+    DISKIMAGE=$TESTDIR/TEST-10-RAID-root.img
     $testdir/run-qemu \
 	-hda $DISKIMAGE \
 	-m 256M -nographic \
@@ -17,6 +17,7 @@ test_run() {
 }
 
 test_setup() {
+    DISKIMAGE=$TESTDIR/TEST-10-RAID-root.img
     # Create the blank file to use as a root filesystem
     rm -f $DISKIMAGE
     dd if=/dev/null of=$DISKIMAGE bs=1M seek=40
