From 7b07f4e83fe514dbbe66869337d3d0cf8610240b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 25 Jul 2011 10:55:19 +0200
Subject: [PATCH] dracut-functions: resolve relative path and recursively
 install destination

---
 dracut-functions |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 2a45419..a261ddd 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -277,12 +277,10 @@ inst_dir() {
         if [[ -L $_file ]]; then
             # create link as the original
             local target=$(readlink "$_file")
+            # resolve relative path and recursively install destination
+            [[ $target == ${target#/} ]] && target="$(dirname "$_file")/$target"
             inst_dir "$target"
             inst_symlink "$_file"
-            #ln -sfn "$target" "${initdir}$_file" || return 1
-            # resolve relative path and recursively install destination
-            #[[ $target == ${target#/} ]] && target="$(dirname "$_file")/$target"
-            #inst_dir "$target"
         else
             [[ -h ${initdir}$_file ]] && _file=$(readlink "${initdir}$_file")
             # create directory
