From 3409c2377fde4e99302a47e23dfe72039d0c941f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 16:47:28 +0200
Subject: [PATCH 25/40] loginit: turn off debugging

---
 modules.d/99base/loginit |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/modules.d/99base/loginit b/modules.d/99base/loginit
index a777d6f..1fb2a92 100755
--- a/modules.d/99base/loginit
+++ b/modules.d/99base/loginit
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# turn off debugging
+set +x
+
 QUIET=$1
 
 [ -e /dev/kmsg ] && exec 5>/dev/kmsg || exec 5>/dev/null
-- 
1.7.0.1

