From 185e940e27eb8c1658556d17b30ac60a16894502 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 4 Nov 2013 17:29:15 +0100
Subject: [PATCH] fips: also install /etc/system-fips in the initramfs

---
 modules.d/01fips/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index fb5a5cd..a4081dc 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -45,5 +45,6 @@ install() {
         libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10
 
     inst_multiple -o prelink
+    inst_simple /etc/system-fips
 }
 
