From 06c6ea654f09c9993aaddab137b4af011d9d6c0e Mon Sep 17 00:00:00 2001
From: Chao Wang <chaowang@redhat.com>
Date: Thu, 26 Jul 2012 11:24:25 +0800
Subject: [PATCH] 95fstab-sys: Skip mounted filesystem.

Signed-off-by: Chao Wang <chaowang@redhat.com>
---
 modules.d/95fstab-sys/mount-sys.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95fstab-sys/mount-sys.sh b/modules.d/95fstab-sys/mount-sys.sh
index a1e5ab1..12711a0 100755
--- a/modules.d/95fstab-sys/mount-sys.sh
+++ b/modules.d/95fstab-sys/mount-sys.sh
@@ -11,6 +11,7 @@ fstab_mount() {
     info "Mounting from $1"
     while read _dev _mp _fs _opts _dump _pass _rest; do
         [ -z "${_dev%%#*}" ] && continue # Skip comment lines
+        ismounted $_mp && continue # Skip mounted filesystem
         if [ "$_pass" -gt 0 ] && ! strstr "$_opts" _netdev; then
             fsck_single "$_dev" "$_fs" "$_opts"
         fi
