From 393da0c370deaea03b7f72a35782fde506526a49 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 14 Aug 2018 11:35:26 +0200
Subject: [PATCH] TEST-50-MULTINIC: s/--device/-device

---
 test/TEST-50-MULTINIC/test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/TEST-50-MULTINIC/test.sh b/test/TEST-50-MULTINIC/test.sh
index de40790f..2640c318 100755
--- a/test/TEST-50-MULTINIC/test.sh
+++ b/test/TEST-50-MULTINIC/test.sh
@@ -57,7 +57,7 @@ client_test() {
       -net nic,macaddr=52:54:00:12:34:$mac3,model=e1000 \
       -netdev hubport,id=n1,hubid=1 \
       -netdev hubport,id=n2,hubid=2 \
-      --device e1000,netdev=n1,mac=52:54:00:12:34:98 \
+      -device e1000,netdev=n1,mac=52:54:00:12:34:98 \
       -device e1000,netdev=n2,mac=52:54:00:12:34:99 \
       -watchdog i6300esb -watchdog-action poweroff \
       -no-reboot \

