From 49fe860b22afd355b67f5f3f2ee30d5a3f0d3f5f Mon Sep 17 00:00:00 2001
From: Nicolas Chauvet <kwizart@gmail.com>
Date: Mon, 4 Aug 2014 22:04:39 +0200
Subject: [PATCH] Introduce ARM specific modules needed by drm

---
 modules.d/50drm/module-setup.sh | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/modules.d/50drm/module-setup.sh b/modules.d/50drm/module-setup.sh
index 9a48aaf..a42bcba 100755
--- a/modules.d/50drm/module-setup.sh
+++ b/modules.d/50drm/module-setup.sh
@@ -51,6 +51,17 @@ installkernel() {
         return 0
     }
 
+    if [[ "$(uname -p)" == arm* ]]; then
+        # arm specific modules needed by drm
+        instmods \
+            "=drivers/gpu/drm/i2c" \
+            "=drivers/gpu/drm/panel" \
+            "=drivers/pwm" \
+            "=drivers/video/backlight" \
+            "=drivers/video/fbdev/omap2/displays-new" \
+            ${NULL}
+    fi
+
     for _modname in $(find_kernel_modules_by_path drivers/gpu/drm \
         | drm_module_filter) ; do
         # if the hardware is present, include module even if it is not currently loaded,
