From 414dc51ca2b60a15405bb2b0b1ac23d4d1ef905c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 4 Dec 2014 11:54:47 +0100
Subject: [PATCH] dracut.conf.d/fedora.conf.example: turn on early_microcode

---
 dracut.conf.d/fedora.conf.example | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.conf.d/fedora.conf.example b/dracut.conf.d/fedora.conf.example
index c06202b..f248f51 100644
--- a/dracut.conf.d/fedora.conf.example
+++ b/dracut.conf.d/fedora.conf.example
@@ -4,6 +4,8 @@
 i18n_vars="/etc/sysconfig/keyboard:KEYTABLE-KEYMAP /etc/sysconfig/i18n:SYSFONT-FONT,FONTACM-FONT_MAP,FONT_UNIMAP"
 i18n_default_font="latarcyrheb-sun16"
 i18n_install_all="yes"
+
+
 stdloglvl=3
 sysloglvl=5
 install_optional_items+=" vi /etc/virc ps grep cat rm "
@@ -13,3 +15,4 @@ systemdsystemunitdir=/usr/lib/systemd/system
 systemdsystemconfdir=/etc/systemd/system
 udevdir=/usr/lib/udev
 hostonly="yes"
+early_microcode="yes"
