From a23b368c9240b8bf64d8163dd092351e0c6cfa29 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Mar 2013 14:45:55 +0100
Subject: [PATCH] systemd: remove upstream renamed old service files

---
 modules.d/98systemd/module-setup.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index bc4f63c..0119291 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -71,14 +71,10 @@ install() {
         $systemdsystemunitdir/systemd-journald.socket \
         $systemdsystemunitdir/systemd-ask-password-console.service \
         $systemdsystemunitdir/systemd-modules-load.service \
-        $systemdsystemunitdir/halt.service \
         $systemdsystemunitdir/systemd-halt.service \
-        $systemdsystemunitdir/poweroff.service \
         $systemdsystemunitdir/systemd-poweroff.service \
         $systemdsystemunitdir/systemd-reboot.service \
-        $systemdsystemunitdir/kexec.service \
         $systemdsystemunitdir/systemd-kexec.service \
-        $systemdsystemunitdir/fsck@.service \
         $systemdsystemunitdir/systemd-fsck@.service \
         $systemdsystemunitdir/systemd-udevd.service \
         $systemdsystemunitdir/systemd-udev-trigger.service \
@@ -97,7 +93,6 @@ install() {
 \
         $systemdsystemunitdir/ctrl-alt-del.target \
         $systemdsystemunitdir/syslog.socket \
-        $systemdsystemunitdir/syslog.target \
         $systemdsystemunitdir/initrd-switch-root.target \
         $systemdsystemunitdir/initrd-switch-root.service \
         $systemdsystemunitdir/initrd-cleanup.service \
