From 08e11e30e328ad2789fd0485dc477a48745af4d6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 7 May 2018 15:27:22 +0200
Subject: [PATCH] dracut.spec: do not recommend grubby

---
 dracut.spec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index d25adcb0..ca8accb5 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -77,7 +77,6 @@ Requires: xz
 Requires: gzip
 
 %if 0%{?fedora} || 0%{?rhel}
-Recommends: grubby
 Recommends: hardlink
 Recommends: pigz
 Recommends: kpartx

