From cebab07e684ef3a051b4fb72a3499fa76cb95b36 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Feb 2013 15:40:47 +0100
Subject: [PATCH] systemd/dracut-initqueue: run after cryptsetup.target

---
 modules.d/98systemd/dracut-initqueue.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/dracut-initqueue.service b/modules.d/98systemd/dracut-initqueue.service
index 3a97bdf..b852468 100644
--- a/modules.d/98systemd/dracut-initqueue.service
+++ b/modules.d/98systemd/dracut-initqueue.service
@@ -13,6 +13,7 @@ Documentation=man:dracut-initqueue.service(8)
 DefaultDependencies=no
 After=systemd-udev-trigger.service
 Wants=systemd-udev-trigger.service
+After=cryptsetup.target
 ConditionPathExists=/etc/initrd-release
 
 [Service]
