From 95957248a18cd1bd87d017b1b9ec81f2e193be45 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 15 Aug 2014 14:56:42 +0200
Subject: [PATCH] 51-dracut-rescue.install: fixed /proc/cmdline read

---
 51-dracut-rescue.install | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/51-dracut-rescue.install b/51-dracut-rescue.install
index b655bc2..677805a 100755
--- a/51-dracut-rescue.install
+++ b/51-dracut-rescue.install
@@ -45,7 +45,7 @@ if [[ -f /etc/kernel/cmdline ]]; then
     readarray -t BOOT_OPTIONS < /etc/kernel/cmdline
 fi
 if ! [[ "${BOOT_OPTIONS[@]}" ]]; then
-    readarray -t BOOT_OPTIONS < /proc/cmdline
+    read -ar BOOT_OPTIONS < /proc/cmdline
 fi
 if ! [[ $BOOT_OPTIONS ]]; then
     exit 1
@@ -75,7 +75,7 @@ case "$COMMAND" in
             echo "Can't copy '$KERNEL_IMAGE to '$BOOT_DIR_ABS/linux'!" >&2
         fi
 
-        dracut --no-hostonly  -a "rescue" "$BOOT_DIR_ABS"/initrd "$2"
+        dracut --no-hostonly -a "rescue" "$BOOT_DIR_ABS"/initrd "$KERNEL_VERSION"
         ((ret+=$?))
 
         {
