From 2663232a018bb3605e1668e51a095205b014fdc9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Feb 2013 14:55:24 +0100
Subject: [PATCH] systemd/dracut-initqueue.sh: call udevadm settle directly

---
 modules.d/98systemd/dracut-initqueue.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/98systemd/dracut-initqueue.sh b/modules.d/98systemd/dracut-initqueue.sh
index 76160cb..7de89e9 100755
--- a/modules.d/98systemd/dracut-initqueue.sh
+++ b/modules.d/98systemd/dracut-initqueue.sh
@@ -25,7 +25,7 @@ while :; do
 
     check_finished && break
 
-    udevsettle
+    udevadm settle --exit-if-exists=$hookdir/initqueue/work
 
     check_finished && break
 
