From 6ecb9a2d68436fd5772d3be0b024852efabe466b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 11 Feb 2013 13:56:04 +0100
Subject: [PATCH] url-lib/url-lib.sh: add proxy support for curl

---
 modules.d/45url-lib/url-lib.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules.d/45url-lib/url-lib.sh b/modules.d/45url-lib/url-lib.sh
index 72c865f..1f8218d 100755
--- a/modules.d/45url-lib/url-lib.sh
+++ b/modules.d/45url-lib/url-lib.sh
@@ -57,6 +57,9 @@ mkdir -p $CURL_HOME
 curl_args="--location --retry 3 --fail --show-error"
 getargbool 0 rd.noverifyssl && curl_args="$curl_args --insecure"
 
+proxy=$(getarg proxy=)
+[ -n "$proxy" ] && curl_args+="$curl_args --proxy $proxy"
+
 curl_fetch_url() {
     local url="$1" outloc="$2"
     echo "$url" > /proc/self/fd/0
