From 03901ba727136d02f76a2ec6ad6cc9a971b1dd10 Mon Sep 17 00:00:00 2001
From: Fedora dracut team <dracut-maint@redhat.com>
Date: Fri, 26 Sep 2014 08:49:24 -0400
Subject: [PATCH] Enable early-microcode by default

See bug 1083716
---
 dracut.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 2cce85b..e4f171b 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -801,7 +801,7 @@ stdloglvl=$((stdloglvl + verbosity_mod_l))
 [[ $nofscks_l ]] && nofscks="yes"
 [[ $ro_mnt_l ]] && ro_mnt="yes"
 [[ $early_microcode_l ]] && early_microcode=$early_microcode_l
-[[ $early_microcode ]] || early_microcode=no
+[[ $early_microcode ]] || early_microcode=yes
 [[ $logfile_l ]] && logfile="$logfile_l"
 [[ $reproducible_l ]] && reproducible="$reproducible_l"
 
-- 
1.9.3

