From b127294def5efecc27fac730f784f8bf03a5e52d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 14 Mar 2016 10:59:29 +0100
Subject: [PATCH] dracut-install: catch ldd message "cannot execute binary
 file"

---
 install/dracut-install.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/install/dracut-install.c b/install/dracut-install.c
index a7bfdb8..8f65d8d 100644
--- a/install/dracut-install.c
+++ b/install/dracut-install.c
@@ -393,6 +393,9 @@ static int resolve_deps(const char *src)
 			break;
 
 		/* glibc */
+                if (strstr(buf, "cannot execute binary file"))
+                        break;
+
                 if (strstr(buf, "not a dynamic executable"))
                         break;
 
