From e38e73d5a683ce93146c147305278b9175def3a0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 2 Dec 2013 10:30:52 +0100
Subject: [PATCH] dracut.cmdline.7.asc: document "server-ip" of "ip=" parameter

---
 dracut.cmdline.7.asc | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/dracut.cmdline.7.asc b/dracut.cmdline.7.asc
index 19bfb93..6a5fc8f 100644
--- a/dracut.cmdline.7.asc
+++ b/dracut.cmdline.7.asc
@@ -444,10 +444,12 @@ cannot be used in conjunction with the **ifname** argument for the
 same <interface>.
 =====================
 
-**ip=**__<client-IP>__:__<server-IP>__:__<gateway-IP>__:__<netmask>__:__<client_hostname>__:__<interface>__:__{none|off|dhcp|on|any|dhcp6|auto6|ibft}__:[:[__<mtu>__][:__<macaddr>__]]::
+**ip=**__<client-IP>__:[__<peer>__]:__<gateway-IP>__:__<netmask>__:__<client_hostname>__:__<interface>__:__{none|off|dhcp|on|any|dhcp6|auto6|ibft}__:[:[__<mtu>__][:__<macaddr>__]]::
     explicit network configuration. If you want do define a IPv6 address, put it
     in brackets (e.g. [2001:DB8::1]). This parameter can be specified multiple
-    times.
+    times. __<peer>__ is optional and is the address of the remote endpoint
+    for pointopoint interfaces and it may be followed by a slash and a decimal number,
+    encoding the network prefix length.
 +
 =====================
 <macaddr>::: optionally **set** <macaddr> on the <interface>. This
