From c390590fac363a3ca467d4be0339baa01936e85d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 14 Mar 2013 17:51:47 +0100
Subject: [PATCH] TODO: remove completed items

---
 TODO | 2 --
 1 file changed, 2 deletions(-)

diff --git a/TODO b/TODO
index 2b7da3f..9eeedfa 100644
--- a/TODO
+++ b/TODO
@@ -24,10 +24,8 @@ GENERATOR TODO
 
 - add presets (predefined set of modules)
 - add interpreter/plugin-scripts to be sourced at the beginning or end (can use dracut-functions)
-- provide "installkernel" and "new-kernel-pkg"
 - add mechanism for module specific command line options
 - pkg-config integration, to make it easy for other packages to use us.
-- add recovery image creator (mkrecovery)
 - default module specification could use some work
 - udev rule copying, as mentioned above, is a bit too hard-coded
 
