From 472928ec3dcbfcc7ea0c1fd7e821843739f03bfc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 14 Mar 2016 14:16:12 +0100
Subject: [PATCH] dracut.conf.5.asc: fix bold

---
 dracut.conf.5.asc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.conf.5.asc b/dracut.conf.5.asc
index e07c247..8f362b2 100644
--- a/dracut.conf.5.asc
+++ b/dracut.conf.5.asc
@@ -187,7 +187,7 @@ provide a valid _/etc/fstab_.
 *show_modules=*"__{yes|no}__"::
     Print the name of the included modules to standard output during build.
 
-*i18n_vars="__<variable mapping>__"::
+*i18n_vars=*"__<variable mapping>__"::
     Distribution specific variable mapping.
     See dracut/modules.d/10i18n/README for a detailed description.
 
