From aaced3f9909dddc0ff858634711c3e97b80b09ec Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 29 Jul 2011 11:12:07 +0200
Subject: [PATCH] git2spec.pl: make rename diffs

---
 git2spec.pl |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/git2spec.pl b/git2spec.pl
index 6e0bff8..8525ad2 100755
--- a/git2spec.pl
+++ b/git2spec.pl
@@ -19,7 +19,7 @@ sub last_tag {
 sub create_patches {
     my $tag=shift;
     my $num=0;
-    open( GIT, 'git format-patch -M -N --no-signature '.$tag.' |');
+    open( GIT, 'git format-patch --no-renames -N --no-signature '.$tag.' |');
     @lines=<GIT>;
     close GIT;         # be done
     return @lines;
