From 15976a06701fc565b591e2bbfc36d0f5c3daac2b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 26 Jul 2012 12:43:20 +0200
Subject: [PATCH] multipath/module-setup.sh: add dm dependency

---
 modules.d/90multipath/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90multipath/module-setup.sh b/modules.d/90multipath/module-setup.sh
index 862a8ed..f9c1f81 100755
--- a/modules.d/90multipath/module-setup.sh
+++ b/modules.d/90multipath/module-setup.sh
@@ -24,6 +24,7 @@ check() {
 
 depends() {
     echo rootfs-block
+    echo dm
     return 0
 }
 
