From 0de93fa1dae15839fc8738268942622ffb4a6451 Mon Sep 17 00:00:00 2001
From: Hermann Gausterer <git-dracut-2012@mrq1.org>
Date: Mon, 16 Jan 2012 17:19:12 +0100
Subject: [PATCH] typo fix

Signed-off-by: Hermann Gausterer <git-dracut-2012@mrq1.org>
---
 modules.d/95iscsi/parse-iscsiroot.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/95iscsi/parse-iscsiroot.sh b/modules.d/95iscsi/parse-iscsiroot.sh
index bc7cc5e..40720e8 100755
--- a/modules.d/95iscsi/parse-iscsiroot.sh
+++ b/modules.d/95iscsi/parse-iscsiroot.sh
@@ -40,7 +40,7 @@ if [ -n "$iscsiroot" ] ; then
     [ -z "$netroot" ]  && netroot=$root
 
     # @deprecated
-    echo "Warning: Argument isciroot is deprecated and might be removed in a future"
+    echo "Warning: Argument iscsiroot is deprecated and might be removed in a future"
     echo "release. See 'man dracut.kernel' for more information."
 
     # Accept iscsiroot argument?
