From fba4d2843382540839d90abbaa2cae0d5bf1c2dd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 14 Aug 2018 16:30:42 +0200
Subject: [PATCH] TEST-50-MULTINIC: fixed server.log

---
 test/TEST-50-MULTINIC/test.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/TEST-50-MULTINIC/test.sh b/test/TEST-50-MULTINIC/test.sh
index d7cec437..0a4d29c2 100755
--- a/test/TEST-50-MULTINIC/test.sh
+++ b/test/TEST-50-MULTINIC/test.sh
@@ -38,6 +38,8 @@ run_server() {
             -net nic,vlan=0,macaddr=52:54:00:12:34:$mac3,model=e1000 \
             -net nic,vlan=1,macaddr=52:54:00:12:34:98,model=e1000 \
             -net nic,vlan=2,macaddr=52:54:00:12:34:99,model=e1000 \
+            ${SERIAL:+-serial "$SERIAL"} \
+            ${SERIAL:--serial file:"$TESTDIR"/server.log} \
             -watchdog i6300esb -watchdog-action poweroff \
             -no-reboot \
             -append "panic=1 loglevel=7 root=/dev/sda rootfstype=ext3 rw console=ttyS0,115200n81 selinux=0" \

