From ebcfda6c61ff37edf465f9147c5f1ea823e05166 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 17:54:29 +0200
Subject: [PATCH 38/40] test/NBD: check for nbd kernel module first

---
 test/TEST-40-NBD/test.sh |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/test/TEST-40-NBD/test.sh b/test/TEST-40-NBD/test.sh
index 8da1ba4..1a46ed1 100755
--- a/test/TEST-40-NBD/test.sh
+++ b/test/TEST-40-NBD/test.sh
@@ -84,6 +84,7 @@ client_test() {
 }
 
 test_run() {
+    modinfo nbd &>/dev/null || { echo "Kernel does not support nbd"; exit 1; }
     if ! run_server; then
 	echo "Failed to start server" 1>&2
 	return 1
@@ -287,6 +288,9 @@ make_server_root() {
 }
 
 test_setup() {
+
+    modinfo nbd &>/dev/null || { echo "Kernel does not support nbd"; exit 1; }
+
     make_encrypted_root || return 1
     make_client_root || return 1
     make_server_root || return 1
-- 
1.7.0.1

