From f48f9341a5a5b7f4e14d4419ce7f969e665d9c42 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 18 Jun 2012 19:20:39 +0200
Subject: [PATCH] disable TEST-16-DMSQUASH for now

---
 test/TEST-16-DMSQUASH/test.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/TEST-16-DMSQUASH/test.sh b/test/TEST-16-DMSQUASH/test.sh
index 9a10058..3816c18 100755
--- a/test/TEST-16-DMSQUASH/test.sh
+++ b/test/TEST-16-DMSQUASH/test.sh
@@ -11,7 +11,7 @@ test_check() {
         echo "python-imgcreate not installed"
 	return 1
     fi
-    return 0
+    return 1
 }
 
 test_run() {
