From a54658c1580e8e80251b6f9d30475495710e146a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 May 2012 11:32:48 +0200
Subject: [PATCH] TODO: update

---
 TODO |    2 ++
 1 Datei geändert, 2 Zeilen hinzugefügt(+)

diff --git a/TODO b/TODO
index 6c27268..81121de 100644
--- a/TODO
+++ b/TODO
@@ -6,6 +6,8 @@ Items are ordered in priority.
 
 INITRAMFS TODO
 
+- fix btrfs subvolume mounting for /usr (fsck)
+- generate systemd unit dracut-initramfs-restore in /run/systemd dynamically
 - put "root=" parsing hooks in separate hook dir
 - call "root=" parsing hooks after getting new rootpath from dhcp
 - put mount hook in main initqueue loop
