From 8f74a60ced3529de92d1ba0b93cb09c6e9ecb5d9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 18:10:47 +0200
Subject: [PATCH 40/40] btfrs: load btrfs module and updated NEWS

---
 NEWS                             |    1 +
 modules.d/90btrfs/40-btrfs.rules |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/NEWS b/NEWS
index a177156..bd6bf32 100644
--- a/NEWS
+++ b/NEWS
@@ -5,6 +5,7 @@ dracut-006
 - dmraid parse different error messages
 - add cdrom polling mechanism for slow cdroms
 - add module btrfs
+- add btrfsctl scan for btrfs multi-devices (raid)
 - teach dmsquash live-root to use rootflags
 - trigger udev with action=add
 - fixed add_drivers handling 
diff --git a/modules.d/90btrfs/40-btrfs.rules b/modules.d/90btrfs/40-btrfs.rules
index 6bc8a14..92c8553 100644
--- a/modules.d/90btrfs/40-btrfs.rules
+++ b/modules.d/90btrfs/40-btrfs.rules
@@ -1,6 +1,7 @@
 SUBSYSTEM!="block", GOTO="btrfs_end"
 ACTION!="add|change", GOTO="btrfs_end"
 
+ENV{ID_FS_TYPE}=="btrfs", RUN+="/sbin/modprobe btrfs"
 ENV{ID_FS_TYPE}=="btrfs", RUN+="/sbin/btrfsctl -A $env{DEVNAME}"
 
 LABEL="btrfs_end"
-- 
1.7.0.1

