From 64ae4e07a4af4db2cdd1009ab484f89b4a6b9cb2 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 2 Dec 2013 11:05:21 +0100
Subject: [PATCH] dracut.sh: skip crypt swaps with password files

---
 dracut.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 177b813..6975d90 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -947,6 +947,8 @@ if [[ $hostonly ]]; then
                         [[ $_mapper = \#* ]] && continue
                         [[ "$_d" -ef /dev/mapper/"$_mapper" ]] || continue
                         [[ "$_o" ]] || _o="$_p"
+                        # skip entries with password files
+                        [[ "$_p" == /* ]] && [[ -f $_p ]] && continue 2
                         # skip mkswap swap
                         [[ $_o == *swap* ]] && continue 2
                     done < /etc/crypttab
