From 221c342d5068c1fa05e38c44b1044a0aacae4a5c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Thu, 26 Jul 2012 15:05:24 +0200
Subject: [PATCH] 98usrmount: [ "x$_usr_found" != "x" ] - removed redundant
 'x'

---
 modules.d/98usrmount/mount-usr.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/98usrmount/mount-usr.sh b/modules.d/98usrmount/mount-usr.sh
index f4d59b3..cca41c3 100755
--- a/modules.d/98usrmount/mount-usr.sh
+++ b/modules.d/98usrmount/mount-usr.sh
@@ -70,7 +70,7 @@ mount_usr()
         fi
     done < "$NEWROOT/etc/fstab" >> /etc/fstab
 
-    if [ "x$_usr_found" != "x" ]; then
+    if [ "$_usr_found" != "" ]; then
         # we have to mount /usr
         _fsck_ret=0
         if ! getargbool 0 rd.skipfsck; then
