From 529f7bf358d30f5e76cd680bd82fc5d954fa8a33 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 21 Jun 2012 01:01:07 +0200
Subject: [PATCH] systemd/module-setup.sh: add old udev systemd services

---
 modules.d/98systemd/module-setup.sh |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index dbb9063..2160eb4 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -102,6 +102,15 @@ install() {
         $systemdsystemunitdir/initrd-switch-root.target \
         $systemdsystemunitdir/initrd-switch-root.service \
         $systemdsystemunitdir/umount.target \
+        $systemdsystemunitdir/udev-control.socket \
+        $systemdsystemunitdir/udev-kernel.socket \
+        $systemdsystemunitdir/udev.service \
+        $systemdsystemunitdir/udev-settle.service \
+        $systemdsystemunitdir/udev-trigger.service \
+        $systemdsystemunitdir/basic.target.wants/udev.service \
+        $systemdsystemunitdir/basic.target.wants/udev-trigger.service \
+        $systemdsystemunitdir/sockets.target.wants/udev-control.socket \
+        $systemdsystemunitdir/sockets.target.wants/udev-kernel.socket
 
     for i in /etc/systemd/*.conf; do
         dracut_install "$i"
