From 6b46e7baa0e16a864259cb9e60aa99e496be6d78 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 14 Mar 2013 19:03:20 +0100
Subject: [PATCH] systemd: do not use "systemd --version" until fixed

---
 modules.d/98systemd/module-setup.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index f74ae88..17dbc44 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -18,11 +18,11 @@ depends() {
 install() {
     local _mods
 
-    SYSTEMD_VERSION=$($systemdutildir/systemd --version | { read a b a; echo $b; })
-    if (( $SYSTEMD_VERSION < 198 )); then
-        dfatal "systemd version $SYSTEMD_VERSION is too low. Need at least version 198."
-        exit 1
-    fi
+#    SYSTEMD_VERSION=$($systemdutildir/systemd --version | { read a b a; echo $b; })
+#    if (( $SYSTEMD_VERSION < 198 )); then
+#        dfatal "systemd version $SYSTEMD_VERSION is too low. Need at least version 198."
+#        exit 1
+#    fi
 
     if strstr "$prefix" "/run/"; then
         dfatal "systemd does not work a prefix, which contains \"/run\"!!"
