From fe98cfee37224420ee133c1b10e7befab24dce05 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 9 May 2012 11:44:23 +0200
Subject: [PATCH] base/dracut-lib.sh: export UDEVVERSION

---
 modules.d/99base/dracut-lib.sh |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index e46e456..2fe8a5e 100755
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -383,7 +383,7 @@ incol2() {
 }
 
 udevsettle() {
-    [ -z "$UDEVVERSION" ] && UDEVVERSION=$(udevadm --version)
+    [ -z "$UDEVVERSION" ] && export UDEVVERSION=$(udevadm --version)
 
     if [ $UDEVVERSION -ge 143 ]; then
         udevadm settle --exit-if-exists=$hookdir/initqueue/work $settle_exit_if_exists
@@ -393,7 +393,7 @@ udevsettle() {
 }
 
 udevproperty() {
-    [ -z "$UDEVVERSION" ] && UDEVVERSION=$(udevadm --version)
+    [ -z "$UDEVVERSION" ] && export UDEVVERSION=$(udevadm --version)
 
     if [ $UDEVVERSION -ge 143 ]; then
         for i in "$@"; do udevadm control --property=$i; done
