From 98fd06934c9e10567b4755714191cd2aee8822ac Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 11 Apr 2016 15:22:11 +0200
Subject: [PATCH] lsinitrd.sh: fixed unpack and skipcpio search

---
 lsinitrd.sh | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/lsinitrd.sh b/lsinitrd.sh
index 224b9c1..b19a66a 100755
--- a/lsinitrd.sh
+++ b/lsinitrd.sh
@@ -40,7 +40,6 @@ usage()
 
 sorted=0
 modules=0
-unpack=0
 unset verbose
 declare -A filenames
 
@@ -184,7 +183,11 @@ case $bin in
                 echo "Early CPIO image"
                 list_files
             fi
-            SKIP="$dracutbasedir/skipcpio"
+            if [[ -d "$dracutbasedir/skipcpio" ]]; then
+                SKIP="$dracutbasedir/skipcpio/skipcpio"
+            else
+                SKIP="$dracutbasedir/skipcpio"
+            fi
             if ! [[ -x $SKIP ]]; then
                 echo
                 echo "'$SKIP' not found, cannot display remaining contents!" >&2
