From 311198868044b79e8e4f37872514f580b51e4ceb Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 16 Aug 2018 09:19:52 +0200
Subject: [PATCH] dracut.sh: remove bogus dir removal for --rebuild

---
 dracut.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/dracut.sh b/dracut.sh
index 6614d27d..8ee8c168 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -474,9 +474,6 @@ if [[ $append_args_l == "yes" ]]; then
         eval set -- "$TEMP"
         rearrange_params "$@"
     fi
-
-    # clean the temporarily used scratch-pad directory
-    rm -rf $scratch_dir
 fi
 
 unset PARMS_TO_STORE

