From 7e9bfdde14e64b959f5ae24aa8e6d91e56d09ec6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 15 Mar 2013 10:41:41 +0100
Subject: [PATCH] sosreport.sh: use -o short-monotonic for the journal output

---
 modules.d/99base/sosreport.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99base/sosreport.sh b/modules.d/99base/sosreport.sh
index 2aded6b..0b4ccdc 100755
--- a/modules.d/99base/sosreport.sh
+++ b/modules.d/99base/sosreport.sh
@@ -41,7 +41,7 @@ command -v dmsetup >/dev/null 2>/dev/null && dmsetup ls --tree
 cat /proc/mdstat
 
 if command -v journalctl >/dev/null 2>/dev/null; then
-    journalctl -ab --no-pager
+    journalctl -ab --no-pager -o short-monotonic
 else
     dmesg
     [ -f /run/initramfs/init.log ] && cat /run/initramfs/init.log
