From 79afb279c273d46b65fafc2c686ed99c5433fe94 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 16 Aug 2018 09:48:02 +0200
Subject: [PATCH] travis: fedora-test.sh send dnf output to dev/null

---
 fedora-test.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fedora-test.sh b/fedora-test.sh
index 5e7d34d6..e7d0f633 100755
--- a/fedora-test.sh
+++ b/fedora-test.sh
@@ -7,7 +7,7 @@ set -ex
 RUN_ID="$1"
 TESTS=$2
 
-dnf -y update --best --allowerasing
+dnf -y update --best --allowerasing &>/dev/null
 
 dnf -y install --best --allowerasing \
     dash \
@@ -39,7 +39,7 @@ dnf -y install --best --allowerasing \
     /usr/bin/qemu-kvm \
     /usr/bin/qemu-system-$(uname -i) \
     e2fsprogs \
-    $NULL
+    $NULL &>/dev/null
 
 ./configure
 

