From 19b5c10963e80cc6899ee4131474c1955754f8c6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 28 May 2013 14:25:49 +0200
Subject: [PATCH] dracut-functions.sh: clarify instmods() error message

---
 dracut-functions.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 96d3c6e..4a0e81a 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -1608,7 +1608,7 @@ instmods() {
             while read _mod; do
                 inst1mod "${_mod%.ko*}" || {
                     if [[ "$_check" == "yes" ]]; then
-                        [[ "$_silent" == "no" ]] && dfatal "Failed to install $_mod"
+                        [[ "$_silent" == "no" ]] && dfatal "Failed to install module $_mod"
                         return 1
                     fi
                 }
@@ -1617,7 +1617,7 @@ instmods() {
         while (($# > 0)); do  # filenames as arguments
             inst1mod ${1%.ko*} || {
                 if [[ "$_check" == "yes" ]]; then
-                    [[ "$_silent" == "no" ]] && dfatal "Failed to install $1"
+                    [[ "$_silent" == "no" ]] && dfatal "Failed to install module $1"
                     return 1
                 fi
             }
