From e42c7a980ca3852781ee2ddd2b1633e482cf58d0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Sat, 16 Mar 2013 10:34:20 +0100
Subject: [PATCH] sosreport: mkdir /run/initramfs, if it does not exist yet

---
 modules.d/99base/sosreport.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/99base/sosreport.sh b/modules.d/99base/sosreport.sh
index 0b4ccdc..eb1ba80 100755
--- a/modules.d/99base/sosreport.sh
+++ b/modules.d/99base/sosreport.sh
@@ -2,6 +2,8 @@
 
 echo 'Generating "/run/initramfs/sosreport.txt"'
 
+[ -d /run/initramfs ] || mkdir -p /run/initramfs
+
 exec >/run/initramfs/sosreport.txt 2>&1
 
 set -x
