From 16c88e493f87aeba449434947a2a1924cf49db43 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 21 Jun 2012 01:09:02 +0200
Subject: [PATCH] dracut.conf.d/fedora.conf.example: make systemd default

---
 dracut.conf.d/fedora.conf.example |    1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.conf.d/fedora.conf.example b/dracut.conf.d/fedora.conf.example
index f3f57f1..25b5cfa 100644
--- a/dracut.conf.d/fedora.conf.example
+++ b/dracut.conf.d/fedora.conf.example
@@ -4,6 +4,7 @@
 i18n_vars="/etc/sysconfig/keyboard:KEYTABLE-KEYMAP /etc/sysconfig/i18n:SYSFONT-FONT,FONTACM-FONT_MAP,FONT_UNIMAP"
 omit_dracutmodules+=" dash "
 omit_drivers+=" .*/fs/ocfs/.* "
+add_dracutmodules+=" systemd "
 stdloglvl=3
 realinitpath="/usr/lib/systemd/systemd"
 install_items+=" vi /etc/virc ps grep cat rm "
