From 6769292c0d43316b116552b0af56509496f96d80 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Jan 2012 11:06:16 +0100
Subject: [PATCH] AUTHORS: updated and fixed .mailmap

---
 .mailmap |    5 ++++-
 AUTHORS  |   28 +++++++++++++++++++---------
 2 files changed, 23 insertions(+), 10 deletions(-)

diff --git a/.mailmap b/.mailmap
index b37295e..bce9b7c 100644
--- a/.mailmap
+++ b/.mailmap
@@ -6,4 +6,7 @@ Harald Hoyer <harald@redhat.com> <harald-H+wXaHxf7aLQT0dZR+AlfA@public.gmane.org
 Harald Hoyer <harald@redhat.com> <harald@eeepc.(none)>
 Mike Snitzer <snitzer@redhat.com> <msnitzer@redhat.com> 
 Amerigo Wang <amwang@redhat.com> <xiyou.wangcong@gmail.com>
-
+Andrey Borzenkov <arvidjaar@gmail.com> <arvidjaar@mail.ru>
+Dan Horák <dhorak@redhat.com> <dan@danny.cz>
+John Reiser <jreiser@bitwagon.com> <jreiser@BitWagon.com>
+Luca Berra <bluca@vodka.it> <bluca@comedia.it>
diff --git a/AUTHORS b/AUTHORS
index 99eab35..12dbe15 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -7,45 +7,55 @@ Jeremy Katz <katzj@redhat.com>
 David Dillow <dave@thedillows.org>
 Michal Soltys <soltys@ziu.info>
 Will Woods <wwoods@redhat.com>
+Andrey Borzenkov <arvidjaar@gmail.com>
 Andreas Thienemann <andreas@bawue.net>
 Hans de Goede <hdegoede@redhat.com>
 Peter Jones <pjones@redhat.com>
-Luca Berra <bluca@comedia.it>
-Andrey Borzenkov <arvidjaar@mail.ru>
-Andrey Borzenkov <arvidjaar@gmail.com>
-Marc Grimme <grimme@atix.de>
+Amerigo Wang <amwang@redhat.com>
+Colin Guthrie <colin@mageia.org>
+John Reiser <jreiser@bitwagon.com>
+Luca Berra <bluca@vodka.it>
+Dave Young <dyoung@redhat.com>
 Daniel Drake <dsd@laptop.org>
+Marc Grimme <grimme@atix.de>
 Roberto Sassu <roberto.sassu@polito.it>
-Amerigo Wang <amwang@redhat.com>
+Anton Blanchard <anton@samba.org>
 Bill Nottingham <notting@redhat.com>
+Brian C. Lane <bcl@redhat.com>
+Dan Horák <dhorak@redhat.com>
 David Cantrell <dcantrell@redhat.com>
 Lance Albertson <lance@osuosl.org>
 Marian Ganisin <mganisin@redhat.com>
 Michael Ploujnikov <plouj@somanetworks.com>
 Alan Jenkins <alan-jenkins@tuffmail.co.uk>
 Alan Pevec <apevec@redhat.com>
-Dan Horák <dan@danny.cz>
+Ian Dall <ian@beware.dropbear.id.au>
+James Buren <ryuo@frugalware.org>
 Joey Boggs <jboggs@redhat.com>
 Jon Ander Hernandez <jonan.h@gmail.com>
 Mike Snitzer <snitzer@redhat.com>
 Peter Rajnoha <prajnoha@redhat.com>
+Przemysław Rudy <prudy1@o2.pl>
 Vladislav Bogdanov <bubble@hoster-ok.com>
 Alexander Todorov <atodorov@redhat.com>
 Andy Lutomirski <luto@mit.edu>
+Anssi Hannula <anssi@mageia.org>
 Christian Heinz <christian.ch.heinz@gmail.com>
-Dan Horák <dhorak@redhat.com>
 Dave Jones <davej@redhat.com>
+Duane Griffin <duaneg@dghda.com>
 Frederic Crozat <fcrozat@mandriva.com>
 Glen Gray <slaine@slaine.org>
-Ian Dall <ian@beware.dropbear.id.au>
+Hermann Gausterer <git-dracut-2012@mrq1.org>
 James Laska <jlaska@redhat.com>
 Konrad Rzeszutek Wilk <konrad.wilk@oracle.com>
+Leho Kraav <leho@kraav.com>
 Lubomir Rintel <lkundrak@v3.sk>
-Luca Berra <bluca@vodka.it>
 Matt <smoothsailing72@hotmail.com>
 Matt Smith <shadowfax@gmx.com>
 Michal Schmidt <mschmidt@redhat.com>
 Munehiro Matsuda <haro@kgt.co.jp>
+Paolo Bonzini <pbonzini@redhat.com>
+Peter Robinson <pbrobinson@fedoraproject.org>
 Pádraig Brady <P@draigBrady.com>
 Quentin Armitage <quentin@armitage.org.uk>
 Sergey Fionov <fionov@gmail.com>
