From 1e2673fd98d05ae5103f24fbd8fb748c9dfc5a45 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 21 Jun 2012 01:20:11 +0200
Subject: [PATCH] dracut.spec: require systemd >= 44-15

---
 dracut.spec |    1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.spec b/dracut.spec
index 6e8c733..982a66d 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -80,6 +80,7 @@ Requires: file
 Requires: udev > 166
 %if 0%{?fedora} || 0%{?rhel} > 6
 Requires: util-linux >= 2.21
+Requires: systemd >= 44-15
 %else
 Requires: util-linux-ng >= 2.21
 %endif
