From 187a8e08d57cb33d51c31e2270374b99e2ad3133 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Thu, 26 Jul 2012 19:04:30 +0200
Subject: [PATCH] gentoo.conf: enable ro_mnt

---
 dracut.conf.d/gentoo.conf.example | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.conf.d/gentoo.conf.example b/dracut.conf.d/gentoo.conf.example
index 411f9a0..6b74954 100644
--- a/dracut.conf.d/gentoo.conf.example
+++ b/dracut.conf.d/gentoo.conf.example
@@ -2,6 +2,7 @@
 # Dracut config file customized for Gentoo Base System release 2
 
 udevdir=/lib/udev
+ro_mnt=yes
 
 #
 # Modules
