From c21c4dc2b469107ac35d8c1157f245965fd55292 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 5 Dec 2013 15:57:42 +0100
Subject: [PATCH] lvm: install thin tools, only when needed in hostonly

---
 modules.d/90lvm/module-setup.sh | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/modules.d/90lvm/module-setup.sh b/modules.d/90lvm/module-setup.sh
index a220a55..1206310 100755
--- a/modules.d/90lvm/module-setup.sh
+++ b/modules.d/90lvm/module-setup.sh
@@ -97,6 +97,11 @@ install() {
 
     inst_libdir_file "libdevmapper-event-lvm*.so"
 
-    inst_multiple -o thin_dump thin_restore thin_check thin_repair
+    if [[ $hostonly ]] && type -P lvs &>/dev/null; then
+        for dev in "${!host_fs_types[@]}"; do
+            if [[ "$(lvs --noheadings -o segtype "$dev" 2>/dev/null)" == *thin* ]] ; then
+                inst_multiple -o thin_dump thin_restore thin_check thin_repair
+            fi
+        done
+    fi
 }
-
