From 65d1a8a4aee600f7d5016382dd351bf9b835327f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 26 Jul 2012 19:12:40 +0200
Subject: [PATCH] dracut-functions.sh:inst_rule_programs() fixed
 IMPORT{program}

---
 dracut-functions.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index fb28bdc..49051fa 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -781,8 +781,8 @@ inst_rule_programs() {
             dracut_install "$_bin"
         done
     fi
-    if grep -qE 'PROGRAM==?"[^ "]+' "$1"; then
-        for _prog in $(grep -E 'IMPORT==?"[^ "]+' "$1" | sed -r 's/.*IMPORT==?"([^ "]+).*/\1/'); do
+    if grep -qE 'IMPORT\{program\}==?"[^ "]+' "$1"; then
+        for _prog in $(grep -E 'IMPORT\{program\}==?"[^ "]+' "$1" | sed -r 's/.*IMPORT\{program\}==?"([^ "]+).*/\1/'); do
             if [ -x ${udevdir}/$_prog ]; then
                 _bin=${udevdir}/$_prog
             else
