From cc2180876488b6e2b49c36317a593cb10e9124a7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 26 Jul 2012 19:35:12 +0200
Subject: [PATCH] dracut-functions.sh:inst_rule_programs() fix error messages

---
 dracut-functions.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 49051fa..9242204 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -757,7 +757,7 @@ inst_rule_programs() {
                 _bin=${udevdir}/$_prog
             else
                 _bin=$(find_binary "$_prog") || {
-                    dinfo "Skipping program $_prog using in udev rule $(${1##*/}) as it cannot be found"
+                    dinfo "Skipping program $_prog using in udev rule ${1##*/} as it cannot be found"
                     continue;
                 }
             fi
@@ -772,7 +772,7 @@ inst_rule_programs() {
                 _bin=${udevdir}/$_prog
             else
                 _bin=$(find_binary "$_prog") || {
-                    dinfo "Skipping program $_prog using in udev rule $(${1##*/}) as it cannot be found"
+                    dinfo "Skipping program $_prog using in udev rule ${1##*/} as it cannot be found"
                     continue;
                 }
             fi
@@ -787,7 +787,7 @@ inst_rule_programs() {
                 _bin=${udevdir}/$_prog
             else
                 _bin=$(find_binary "$_prog") || {
-                    dinfo "Skipping program $_prog using in udev rule $(${1##*/}) as it cannot be found"
+                    dinfo "Skipping program $_prog using in udev rule ${1##*/} as it cannot be found"
                     continue;
                 }
             fi
