From ce052202f0cdb673536999a7d7810bd3a3a2aaca Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 5 Dec 2013 18:38:23 +0100
Subject: [PATCH] TODO: update

---
 TODO | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TODO b/TODO
index 4e52fbf..875541d 100644
--- a/TODO
+++ b/TODO
@@ -22,6 +22,7 @@ INITRAMFS TODO
 
 GENERATOR TODO
 
+- report errors on missing files in check()
 - remove wait for swap devs, if no "resume=" is given on the kernel command line
 - add presets (predefined set of modules)
 - add interpreter/plugin-scripts to be sourced at the beginning or end (can use dracut-functions)
