From 4f10ae2b86af35051ec07e8710578501d1cf1e0a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Jan 2012 13:49:39 +0100
Subject: [PATCH] dracut: _get_fs_type() also handle /dev/block/maj:min

---
 dracut |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/dracut b/dracut
index f2e5feb..db97b58 100755
--- a/dracut
+++ b/dracut
@@ -574,6 +574,10 @@ _get_fs_type() (
         echo "$1|$ID_FS_TYPE"
         return 1
     fi
+    if [[ -b /dev/block/$1 ]] && get_fs_env /dev/block/$1; then
+        echo "/dev/block/$1|$ID_FS_TYPE"
+        return 1
+    fi
     if fstype=$(find_dev_fstype $1); then
         echo "$1|$fstype"
         return 1
