From 72ee74b9be0935f9421a4987e9de785b74e71a50 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 27 Jul 2012 06:26:26 +0200
Subject: [PATCH] dracut.spec: add cifs module

---
 dracut.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.spec b/dracut.spec
index d960506..f225bb6 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -328,6 +328,7 @@ rm -rf $RPM_BUILD_ROOT
 %{dracutlibdir}/modules.d/95iscsi
 %{dracutlibdir}/modules.d/90livenet
 %{dracutlibdir}/modules.d/90qemu-net
+%{dracutlibdir}/modules.d/95cifs
 %{dracutlibdir}/modules.d/95nbd
 %{dracutlibdir}/modules.d/95nfs
 %{dracutlibdir}/modules.d/95ssh-client
