From af2fcbafe8bdb30c34f440b76c37d535016d739f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 19 Mar 2013 14:52:25 +0100
Subject: [PATCH] network/netroot.sh: do not unset root

---
 modules.d/40network/netroot.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/modules.d/40network/netroot.sh b/modules.d/40network/netroot.sh
index 8c7716b..9c481a8 100755
--- a/modules.d/40network/netroot.sh
+++ b/modules.d/40network/netroot.sh
@@ -24,9 +24,6 @@ netif=$1
 # handlers when this is not called from manually network bringing up.
 if [ -z "$2" ]; then
     if [ "$netroot" = "dhcp" ] || [ "$netroot" = "dhcp6" ] ; then
-        # Unset root so we can check later
-        unset root
-
         # Load dhcp options
         [ -e /tmp/dhclient.$netif.dhcpopts ] && . /tmp/dhclient.$netif.dhcpopts
 
