From c9baff669d659998edb5d8269ef8e004aaad1d4a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Fri, 27 Jul 2012 09:34:32 +0200
Subject: [PATCH] Makefile: add dependencies for dracut-install (generated
 with gcc -MM)

---
 Makefile | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 28381df..cd15009 100644
--- a/Makefile
+++ b/Makefile
@@ -41,6 +41,14 @@ DRACUT_INSTALL_OBJECTS = \
         install/log.o \
         install/util.o
 
+# deps generated with gcc -MM
+install/dracut-install.o: install/dracut-install.c install/log.h install/macro.h \
+	install/hashmap.h install/util.h
+install/hashmap.o: install/hashmap.c install/util.h install/macro.h install/log.h \
+	install/hashmap.h
+install/log.o: install/log.c install/log.h install/macro.h install/util.h
+install/util.o: install/util.c install/util.h install/macro.h install/log.h
+
 install/dracut-install: $(DRACUT_INSTALL_OBJECTS)
 
 dracut-install: install/dracut-install
@@ -109,7 +117,7 @@ clean:
 	$(RM) */*/*~
 	$(RM) test-*.img
 	$(RM) dracut-*.rpm dracut-*.tar.bz2
-	$(RM) $(DRACUT_INSTALL_BIN) install/dracut-install $(DRACUT_INSTALL_OBJECTS)
+	$(RM) dracut-install install/dracut-install $(DRACUT_INSTALL_OBJECTS)
 	$(RM) $(manpages) dracut.html
 	$(MAKE) -C test clean
 
