From 57aa1e91400ff06022afe7175f5b4e97e813cf8d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 19 Mar 2013 14:53:17 +0100
Subject: [PATCH] TEST-40-NBD/test.sh: kill_server() after test_run()

---
 test/TEST-40-NBD/test.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/TEST-40-NBD/test.sh b/test/TEST-40-NBD/test.sh
index dae65f2..4feb557 100755
--- a/test/TEST-40-NBD/test.sh
+++ b/test/TEST-40-NBD/test.sh
@@ -99,7 +99,8 @@ test_run() {
         echo "Failed to start server" 1>&2
         return 1
     fi
-    client_run || { kill_server; return 1; }
+    client_run
+    kill_server
 }
 
 client_run() {
