From b1b704bd67d0b397585618affcf12ad30ceae183 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 12 Aug 2015 14:24:45 +0200
Subject: [PATCH] systemd: add systemd-run and systemd-escape

---
 modules.d/00systemd/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index e763d9f..da95aee 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -129,6 +129,7 @@ install() {
         echo swapoff \
         kmod insmod rmmod modprobe modinfo depmod lsmod \
         mount umount reboot poweroff \
+        systemd-run systemd-escape \
         systemd-cgls systemd-tmpfiles
 
     inst_multiple -o \
