From 8099e7b1ba10eaf7780e6423e97a13a5c62cf5bd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 29 May 2013 13:31:54 +0200
Subject: [PATCH] crypt-loop/module-setup.sh: install loop kernel module

---
 modules.d/91crypt-loop/module-setup.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules.d/91crypt-loop/module-setup.sh b/modules.d/91crypt-loop/module-setup.sh
index 5e7bcbd..218d27a 100644
--- a/modules.d/91crypt-loop/module-setup.sh
+++ b/modules.d/91crypt-loop/module-setup.sh
@@ -8,6 +8,10 @@ depends() {
 	echo crypt
 }
 
+installkernel() {
+	    instmods loop
+}
+
 install() {
 	dracut_install losetup
 	inst "$moddir/crypt-loop-lib.sh" "/lib/dracut-crypt-loop-lib.sh"
