From 82b1739dcee8853c07675d3e79b2f4623c60a364 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 27 Jul 2012 11:54:24 +0200
Subject: [PATCH] cifs/parse-cifsroot.sh: do not unset netroot, we need it

---
 modules.d/95cifs/parse-cifsroot.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95cifs/parse-cifsroot.sh b/modules.d/95cifs/parse-cifsroot.sh
index d5adb98..de7637f 100755
--- a/modules.d/95cifs/parse-cifsroot.sh
+++ b/modules.d/95cifs/parse-cifsroot.sh
@@ -35,7 +35,7 @@ fi
 # Continue if cifs
 case "${netroot%%:*}" in
     cifs);;
-    *) unset netroot; return;;
+    *) return;;
 esac
 
 # Check required arguments
