From 68cbbae33adc65452ee234940f668151f65cc043 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 5 Jan 2018 10:39:44 +0100
Subject: [PATCH] dracut.spec: remove systemd-bootchart

---
 dracut.spec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index 03363fb2..50d42a31 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -371,7 +371,6 @@ rm -rf -- $RPM_BUILD_ROOT
 %endif
 
 %if %{defined _unitdir}
-%{dracutlibdir}/modules.d/00systemd-bootchart
 %else
 %{dracutlibdir}/modules.d/00bootchart
 %endif

