From 286685cb720f4150f363487c3702ea12d838f7df Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 16 Aug 2018 13:20:40 +0200
Subject: [PATCH] TEST-99-RPM: use releasever of the host system

---
 test/TEST-99-RPM/test.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/TEST-99-RPM/test.sh b/test/TEST-99-RPM/test.sh
index 984b9385..b4aca176 100755
--- a/test/TEST-99-RPM/test.sh
+++ b/test/TEST-99-RPM/test.sh
@@ -30,12 +30,12 @@ test_run() {
        "$TESTDIR"/dracut-[0-9]*.$(arch).rpm \
        "$TESTDIR"/dracut-network-[0-9]*.$(arch).rpm \
        "$rootdir/$TESTDIR/"
-
+    . /etc/os-release
     dnf_or_yum=yum
     dnf_or_yum_cmd=yum
     command -v dnf >/dev/null && { dnf_or_yum="dnf"; dnf_or_yum_cmd="dnf --allowerasing"; }
     for (( i=0; i < 5 ; i++)); do
-        $dnf_or_yum_cmd -v --nogpgcheck --installroot "$rootdir"/ --disablerepo='*' \
+        $dnf_or_yum_cmd -v --nogpgcheck --installroot "$rootdir"/ --releasever "$VERSION_ID" --disablerepo='*' \
                         --enablerepo=fedora --enablerepo=updates \
                         install -y \
                         $dnf_or_yum \

