From 2d674a93364d432fbf62988e2fb3e3eda9a3be8f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 27 Jul 2012 12:12:09 +0200
Subject: [PATCH] README.testsuite: add instructions on how to run the
 testsuite

---
 README.testsuite | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/README.testsuite b/README.testsuite
index cbc2ab7..e463c41 100644
--- a/README.testsuite
+++ b/README.testsuite
@@ -10,3 +10,21 @@ nbd
 dhcp
 iscsi-initiator-utils
 TEST-04-FULL-SYSTEMD: systemd >= 187
+
+How to run the testsuite:
+
+$ sudo make clean check
+
+in verbose mode:
+$ sudo make V=1 clean check
+
+only specific test:
+$ sudo make TESTS="01 20 40" clean check
+only runs the 01, 20 and 40 tests.
+
+debug a specific test case:
+$ cd TEST-01-BASIC
+$ sudo make clean setup run
+... change some kernel parameters ...
+$ sudo make run
+to run the test without doing the setup
