From 08f06c93419aacac9d9980fa9c0c65d15fa4bf0e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 16 Dec 2013 12:30:26 +0100
Subject: [PATCH] dracut.spec: add 95fcoe-uefi and 99uefi-lib

---
 dracut.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index bd35aa7..6cc60e1 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -414,6 +414,8 @@ rm -rf -- $RPM_BUILD_ROOT
 %{dracutlibdir}/modules.d/95ssh-client
 %{dracutlibdir}/modules.d/45ifcfg
 %{dracutlibdir}/modules.d/95znet
+%{dracutlibdir}/modules.d/95fcoe-uefi
+%{dracutlibdir}/modules.d/99uefi-lib
 
 %if 0%{?fedora} || 0%{?rhel} || 0%{?suse_version}
 %files fips
