From ae753baec0ae5b3c3eeabbe4f70c54cc3c24c292 Mon Sep 17 00:00:00 2001
From: Ruben Kerkhof <ruben@rubenkerkhof.com>
Date: Fri, 29 Apr 2016 15:34:47 +0200
Subject: [PATCH] Fix small typo in dracut.cmdline(7)

---
 dracut.cmdline.7.asc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.cmdline.7.asc b/dracut.cmdline.7.asc
index c4c5588..bf00719 100644
--- a/dracut.cmdline.7.asc
+++ b/dracut.cmdline.7.asc
@@ -32,7 +32,7 @@ line is the value, which is honored.
 Standard
 ~~~~~~~~
 **init=**__<path to real init>__::
-    specify the path to the init programm to be started after the initramfs has
+    specify the path to the init program to be started after the initramfs has
     finished
 
 **root=**__<path to blockdevice>__::
