From 764eb40cc933d111ad99d74d61e9203f20051e81 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 27 Jul 2012 16:03:25 +0200
Subject: [PATCH] TODO: update

---
 TODO | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TODO b/TODO
index 61be5d5..d74577f 100644
--- a/TODO
+++ b/TODO
@@ -6,6 +6,7 @@ Items are ordered in priority.
 
 INITRAMFS TODO
 
+- use systemd crypto passwd mechanisms
 - add sosreport
 - generate systemd unit dracut-initramfs-restore in /run/systemd dynamically
 - put "root=" parsing hooks in separate hook dir
