From 9918afd244cbee479af4380c1fa2a7a81fba6bed Mon Sep 17 00:00:00 2001
From: Lidong Zhong <lzhong@suse.com>
Date: Mon, 24 Aug 2015 18:02:10 +0800
Subject: [PATCH] dracut-functions.sh: remove duplicate declaratio of local
 variable

---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index d559903..cb88078 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -429,7 +429,7 @@ shorten_persistent_dev() {
 # $ find_block_device /usr
 # 8:4
 find_block_device() {
-    local _majmin _dev _majmin _find_mpt
+    local _dev _majmin _find_mpt
     _find_mpt="$1"
     if [[ $use_fstab != yes ]]; then
         [[ -d $_find_mpt/. ]]
