From 3e964eeb9674119d7c08ae0b1aaee76391a2ba93 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 27 Jul 2012 16:06:36 +0200
Subject: [PATCH] dracut.sh: put $drivers and $filesystems back in export

---
 dracut.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 4be4306..1bbec3d 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -787,7 +787,7 @@ export initdir dracutbasedir dracutmodules \
     stdloglvl sysloglvl fileloglvl kmsgloglvl logfile \
     debug host_fs_types host_devs sshkey add_fstab \
     DRACUT_VERSION udevdir systemdutildir systemdsystemunitdir \
-    prefix
+    prefix filesystems drivers
 
 # Create some directory structure first
 [[ $prefix ]] && mkdir -m 0755 -p "${initdir}${prefix}"
