From 3e7a05f2a5f051a97ac08885f5288bc310c367c2 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 19 Dec 2014 12:47:49 +0100
Subject: [PATCH] systemd: add systemd-journald-audit.socket

---
 modules.d/98systemd/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index 53e07a2..51ea288 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -83,6 +83,7 @@ install() {
         $systemdsystemunitdir/systemd-udevd-kernel.socket \
         $systemdsystemunitdir/systemd-ask-password-plymouth.path \
         $systemdsystemunitdir/systemd-journald.socket \
+        $systemdsystemunitdir/systemd-journald-audit.socket \
         $systemdsystemunitdir/systemd-ask-password-console.service \
         $systemdsystemunitdir/systemd-modules-load.service \
         $systemdsystemunitdir/systemd-halt.service \
@@ -105,6 +106,7 @@ install() {
         $systemdsystemunitdir/sockets.target.wants/systemd-udevd-control.socket \
         $systemdsystemunitdir/sockets.target.wants/systemd-udevd-kernel.socket \
         $systemdsystemunitdir/sockets.target.wants/systemd-journald.socket \
+        $systemdsystemunitdir/sockets.target.wants/systemd-journald-audit.socket \
         $systemdsystemunitdir/sockets.target.wants/systemd-journald-dev-log.socket \
         $systemdsystemunitdir/sysinit.target.wants/systemd-udevd.service \
         $systemdsystemunitdir/sysinit.target.wants/systemd-udev-trigger.service \
