From ea877ac6cc3df38d22728437b8a70bedee558f6a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 3 Sep 2015 12:31:34 +0200
Subject: [PATCH] dracut-functions.sh: fixed dracutbasedir, when sourced
 directly

---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index d559903..3982fc8 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -96,7 +96,7 @@ find_binary() {
 
 if ! [[ $dracutbasedir ]]; then
     dracutbasedir=${BASH_SOURCE[0]%/*}
-    [[ $dracutbasedir = "dracut-functions" ]] && dracutbasedir="."
+    [[ $dracutbasedir = dracut-functions* ]] && dracutbasedir="."
     [[ $dracutbasedir ]] || dracutbasedir="."
     dracutbasedir="$(readlink -f $dracutbasedir)"
 fi
