From dfbb922ddbeceb90f09e4fe20e86a02be7d58fbc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 2 Feb 2012 18:25:49 +0100
Subject: [PATCH] dracut-functions: install nosegneg libs additionally to
 standard ones

---
 dracut-functions |   12 +++++-------
 1 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index e04b16d..b6ac9ac 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -464,18 +464,16 @@ inst_binary() {
 
         # See if we are loading an optimized version of a shared lib.
         if [[ $_file =~ $_lib_regex ]]; then
-            _tlibdir=${BASH_REMATCH[1]}
-            _base=${_file##*/}
-            # Prefer nosegneg libs to unoptimized ones.
+           _tlibdir=${BASH_REMATCH[1]}
+           _base=${_file##*/}
+           # Prefer nosegneg libs to unoptimized ones.
             for _f in "$_tlibdir/i686/nosegneg" "$_tlibdir"; do
                 [[ -e $_f/$_base ]] || continue
-                _file=$_f/$_base
+                inst_library $_f/$_base
                 break
             done
-            inst_library "$_file" "$_tlibdir/$_base"
-        else
-            inst_library "$_file"
         fi
+        inst_library "$_file"
     done
 
     # Install the binary if it wasn't handled in the above loop.
