From b2d225a6698607bbcb73b6e6d12fbfc5be014102 Mon Sep 17 00:00:00 2001
From: Chao Wang <chaowang@redhat.com>
Date: Mon, 30 Jul 2012 11:56:29 +0800
Subject: [PATCH] dracut-functions: $_x should exist, no need to test.

Signed-off-by: Chao Wang <chaowang@redhat.com>
---
 dracut-functions.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 9242204..04b4c7a 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -423,7 +423,6 @@ check_block_and_slaves() {
     fi
     [[ -d /sys/dev/block/$2/slaves ]] || return 1
     for _x in /sys/dev/block/$2/slaves/*/dev; do
-        [[ -f $_x ]] || continue
         check_block_and_slaves $1 $(cat "$_x") && return 0
     done
     return 1
