From 07a8e55f418f9362f074eb4de59070e297f54fd6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 22 Jun 2012 15:23:56 +0200
Subject: [PATCH] bootchart/module-setup.sh: no need for mknod anymore

---
 modules.d/00bootchart/module-setup.sh |    2 --
 1 file changed, 2 deletions(-)

diff --git a/modules.d/00bootchart/module-setup.sh b/modules.d/00bootchart/module-setup.sh
index 17dd137..a2a2c00 100755
--- a/modules.d/00bootchart/module-setup.sh
+++ b/modules.d/00bootchart/module-setup.sh
@@ -25,7 +25,5 @@ install() {
     inst /bin/grep
     inst /bin/usleep
     inst /usr/bin/[  /bin/[
-
-    mknod -m 0666 "${initdir}/dev/null" c 1 3
 }
 
