From 820f81016a31f069dc8ef9d7a6b65ffc1166daa5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 9 Sep 2015 11:30:52 +0200
Subject: [PATCH] dracut.8.asc: mention lsinitrd(1) in see also

---
 dracut.8.asc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.8.asc b/dracut.8.asc
index 7a7ef9d..1fbbda4 100644
--- a/dracut.8.asc
+++ b/dracut.8.asc
@@ -536,4 +536,4 @@ Will Woods
 
 SEE ALSO
 --------
-*dracut.cmdline*(7) *dracut.conf*(5)
+*dracut.cmdline*(7) *dracut.conf*(5) *lsinitrd*(1)
