From 3847bd450d82fd4f95d9668f4810f31adc95ab30 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 8 Jun 2016 16:46:42 +0200
Subject: [PATCH] systemd: add udev.hwdb and udev.conf

---
 modules.d/00systemd/module-setup.sh | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index 951a5be..dcee08b 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -130,7 +130,9 @@ install() {
         kmod insmod rmmod modprobe modinfo depmod lsmod \
         mount umount reboot poweroff \
         systemd-run systemd-escape \
-        systemd-cgls systemd-tmpfiles
+        systemd-cgls systemd-tmpfiles \
+        /etc/udev/udev.hwdb \
+        ${NULL}
 
     inst_multiple -o \
         /usr/lib/modules-load.d/*.conf \
@@ -167,7 +169,9 @@ install() {
             /etc/locale.conf \
             /etc/modules-load.d/*.conf \
             /etc/sysctl.d/*.conf \
-            /etc/sysctl.conf
+            /etc/sysctl.conf \
+            /etc/udev/udev.conf \
+            ${NULL}
 
         _mods=$(modules_load_get /etc/modules-load.d)
         [[ $_mods ]] && instmods $_mods
@@ -196,7 +200,8 @@ install() {
         71-seat.rules \
         73-seat-late.rules \
         90-vconsole.rules \
-        99-systemd.rules
+        99-systemd.rules \
+        ${NULL}
 
     for i in \
         emergency.target \
