From 343b7d7488e7e91a91a0056c4dde2bb57ea203a0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 30 Jul 2012 14:50:14 +0200
Subject: [PATCH] kernel-modules/module-setup.sh: add missing "fi"

---
 modules.d/90kernel-modules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 47e2f0d..4e75ef8 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -61,6 +61,7 @@ installkernel() {
             }
             for_each_host_dev_fs inst_fs
         fi
+    fi
 }
 
 install() {
