From 9befacf0c51e15fe8a199242c3bfd97d1adf455c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 30 Sep 2015 14:10:20 +0200
Subject: [PATCH] dracut.spec: omit pkg-config from dependencies

---
 dracut.spec | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index 6ef5de7..d0bb9bd 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -1,6 +1,11 @@
 %define dracutlibdir %{_prefix}/lib/dracut
 %bcond_without doc
 
+# We ship a .pc file but don't want to have a dep on pkg-config. We
+# strip the automatically generated dep here and instead co-own the
+# directory.
+%global __requires_exclude pkg-config
+
 # Variables must be defined
 %define with_nbd                1
 
@@ -327,6 +332,7 @@ rm -rf -- $RPM_BUILD_ROOT
 %endif
 %dir %{_sysconfdir}/dracut.conf.d
 %dir %{dracutlibdir}/dracut.conf.d
+%dir %{_datadir}/pkgconfig
 %{_datadir}/pkgconfig/dracut.pc
 
 %if %{with doc}
