From 474cca1e9dba2ff185062943d28a0223a7e53f58 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 30 Jul 2012 21:07:03 +0200
Subject: [PATCH] TODO: update

---
 TODO | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TODO b/TODO
index d74577f..9481df7 100644
--- a/TODO
+++ b/TODO
@@ -6,7 +6,7 @@ Items are ordered in priority.
 
 INITRAMFS TODO
 
-- use systemd crypto passwd mechanisms
+- use info and warn prefix
 - add sosreport
 - generate systemd unit dracut-initramfs-restore in /run/systemd dynamically
 - put "root=" parsing hooks in separate hook dir
