From 1c4a51c39c8ed9908ae8118eff78d3476eaddf61 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 31 Aug 2018 16:51:38 +0200
Subject: [PATCH] network: fix an error message

---
 modules.d/40network/netroot.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/netroot.sh b/modules.d/40network/netroot.sh
index c80a59fe..207364cd 100755
--- a/modules.d/40network/netroot.sh
+++ b/modules.d/40network/netroot.sh
@@ -46,7 +46,7 @@ if [ -z "$2" ]; then
         # we die. Otherwise we just warn
         if [ -z "$new_root_path" ] ; then
             [ -n "$BOOTDEV" ] && die "No dhcp root-path received for '$BOOTDEV'"
-            warn "No dhcp root-path received for '$BOOTDEV' trying other interfaces if available"
+            warn "No dhcp root-path received for '$netif' trying other interfaces if available"
             exit 1
         fi
 

