From b94ae0561f4a431a33af167837b84dcb4dd06bde Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 17 Jul 2012 11:09:51 +0200
Subject: [PATCH] dracut.conf.d/fedora.conf.example: disable systemd for now

---
 dracut.conf.d/fedora.conf.example |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.conf.d/fedora.conf.example b/dracut.conf.d/fedora.conf.example
index 4cb1890..012bdd6 100644
--- a/dracut.conf.d/fedora.conf.example
+++ b/dracut.conf.d/fedora.conf.example
@@ -9,5 +9,5 @@ prefix="/"
 systemdutildir=/usr/lib/systemd
 systemdsystemunitdir=/usr/lib/systemd/system
 udevdir=/usr/lib/udev
-add_dracutmodules+=" systemd "
+#add_dracutmodules+=" systemd "
 realinitpath="/usr/lib/systemd/systemd"
