From 2cc5e92e210d8eba1bce3068ca2be96b2427862d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 11 Nov 2015 11:49:34 +0100
Subject: [PATCH] dracut.sh: quote $outfile on error removal

---
 dracut.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 9ba481b..1169037 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1690,9 +1690,10 @@ if ! (
             | $compress >> "$outfile"
     ); then
     dfatal "dracut: creation of $outfile failed"
-	rm -f $outfile
+    rm -f "$outfile"
     exit 1
 fi
+
 dinfo "*** Creating initrd image file '$outfile' done ***"
 
 if (( maxloglvl >= 5 )); then
