From 4be697e5c19f18a17ec18368ded0787d6f32baf5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 21 Sep 2018 12:41:00 +0200
Subject: [PATCH] dracut.spec: fixed invalid spec line

---
 dracut.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index b6099824..c6fbdc77 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -49,7 +49,7 @@ BuildRequires: docbook-style-xsl docbook-dtds libxslt
 %endif
 
 %if 0%{?suse_version}
--BuildRequires: docbook-xsl-stylesheets libxslt
+BuildRequires: docbook-xsl-stylesheets libxslt
 %endif
 
 BuildRequires: asciidoc
