From aaaf035aedd043302c6fec57ba9570afe7639aab Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 20 Jul 2016 18:23:44 +0900
Subject: [PATCH] dracut.conf.5.asc: remove duplicate i18n_install_all option

---
 dracut.conf.5.asc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/dracut.conf.5.asc b/dracut.conf.5.asc
index 8f362b2..318024a 100644
--- a/dracut.conf.5.asc
+++ b/dracut.conf.5.asc
@@ -99,9 +99,6 @@ Configuration files must have the extension .conf; other extensions are ignored.
 *hostonly_cmdline=*"__{yes|no}__"::
     If set, store the kernel command line arguments needed in the initramfs
 
-*i18n_install_all=*"__{yes|no}__"::
-    If set to yes, install all available fonts and keyboard files.
-
 *persistent_policy=*"__<policy>__"::
     Use _<policy>_ to address disks and partitions.
     _<policy>_ can be any directory name found in /dev/disk.
