From a1ea214a874514401c318551b93c9ae06be1747d Mon Sep 17 00:00:00 2001
From: Brendan Germain <brendan.germain@nasdaqomx.com>
Date: Mon, 19 Sep 2016 13:41:32 -0400
Subject: [PATCH] add imjournal.so to read systemd journal

---
 modules.d/98syslog/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/98syslog/module-setup.sh b/modules.d/98syslog/module-setup.sh
index e18aecb..aa0c8ea 100755
--- a/modules.d/98syslog/module-setup.sh
+++ b/modules.d/98syslog/module-setup.sh
@@ -17,7 +17,7 @@ install() {
     local _installs
     if type -P rsyslogd >/dev/null; then
         _installs="rsyslogd"
-        inst_libdir_file rsyslog/lmnet.so rsyslog/imklog.so rsyslog/imuxsock.so
+        inst_libdir_file rsyslog/lmnet.so rsyslog/imklog.so rsyslog/imuxsock.so rsyslog/imjournal.so
     elif type -P syslogd >/dev/null; then
         _installs="syslogd"
     elif type -P syslog-ng >/dev/null; then
