From ca31a7e368f839bc96fc8170c5454d8dc63f74e8 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 3 Nov 2016 15:05:36 +0100
Subject: [PATCH] TODO: remove some items

---
 TODO | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/TODO b/TODO
index 55fe13b..f4fc208 100644
--- a/TODO
+++ b/TODO
@@ -18,11 +18,8 @@ INITRAMFS TODO
 - put mount hook in main initqueue loop / careful about resume!
 - the hard-coded list of udev rules that we care about is kind of lame.
 - panic fallback
-- bridging/bonding without "netroot=" https://bugzilla.redhat.com/show_bug.cgi?id=822750
 - progress indication for fsck https://bugzilla.redhat.com/show_bug.cgi?id=827118
 - domain, searchdomain https://bugzilla.redhat.com/show_bug.cgi?id=840778
-- disable write-ifcfg https://bugzilla.redhat.com/show_bug.cgi?id=840784
-- check for /var to be mounted in convertfs https://bugzilla.redhat.com/show_bug.cgi?id=848172
 - probably fix "--include" https://bugzilla.redhat.com/show_bug.cgi?id=849338
 
 GENERATOR TODO
