From caa6d12fd7cce4704916c29c557bbb1bb592e024 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 7 Nov 2016 17:20:43 +0100
Subject: [PATCH] README.testsuite: add more requirements

---
 README.testsuite | 19 +++++++++++++++++--
 1 file changed, 17 insertions(+), 2 deletions(-)

diff --git a/README.testsuite b/README.testsuite
index f9f60d4..127e302 100644
--- a/README.testsuite
+++ b/README.testsuite
@@ -9,8 +9,23 @@ nfs-utils \
 nbd \
 dhcp-server \
 scsi-target-utils \
-iscsi-initiator-utils
-
+iscsi-initiator-utils \
+net-tools \
+strace \
+syslinux \
+python-imgcreate \
+genisoimage \
+btrfs-progs \
+bridge-utils \
+kmod-devel \
+gcc \
+bzip2 \
+xz \
+tar \
+wget \
+rpm-build \
+dash \
+${NULL}
 
 TEST-04-FULL-SYSTEMD: systemd >= 187
 
