From 0eb7a7c4798b34e42e82ba9ec6a2d0431a97d13a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 10 Nov 2016 10:56:06 +0100
Subject: [PATCH] README.testsuite: add dmraid

---
 README.testsuite | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.testsuite b/README.testsuite
index 127e302..dc08df3 100644
--- a/README.testsuite
+++ b/README.testsuite
@@ -4,6 +4,7 @@ bridge-utils \
 asciidoc \
 mdadm \
 lvm2 \
+dmraid \
 cryptsetup \
 nfs-utils \
 nbd \
@@ -24,7 +25,6 @@ xz \
 tar \
 wget \
 rpm-build \
-dash \
 ${NULL}
 
 TEST-04-FULL-SYSTEMD: systemd >= 187
