From e35d1451f59e35c8bb03172749c331426d65c22f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 11 Nov 2016 16:34:42 +0100
Subject: [PATCH] test/TEST-99-RPM: dnf does not like any dracut subpackage

---
 test/TEST-99-RPM/test.sh | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/test/TEST-99-RPM/test.sh b/test/TEST-99-RPM/test.sh
index 1485278..345d524 100755
--- a/test/TEST-99-RPM/test.sh
+++ b/test/TEST-99-RPM/test.sh
@@ -35,7 +35,7 @@ trap '[[ -d $rootdir ]] && { umount "$rootdir/proc"; umount "$rootdir/sys"; umou
     dnf_or_yum_cmd=yum
     command -v dnf >/dev/null && { dnf_or_yum="dnf"; dnf_or_yum_cmd="dnf --allowerasing"; }
     $dnf_or_yum_cmd -v --nogpgcheck --installroot "$rootdir"/ --releasever 25 --disablerepo='*' \
-                --enablerepo=fedora \
+                --enablerepo=fedora --enablerepo=updates \
                 install -y \
 	$dnf_or_yum \
 	passwd \
@@ -51,8 +51,10 @@ trap '[[ -d $rootdir ]] && { umount "$rootdir/proc"; umount "$rootdir/sys"; umou
     bash \
     iscsi-initiator-utils \
     "$TESTDIR"/dracut-[0-9]*.$(arch).rpm \
-    "$TESTDIR"/dracut-network-[0-9]*.$(arch).rpm \
     ${NULL}
+    #"$TESTDIR"/dracut-config-rescue-[0-9]*.$(arch).rpm \
+    #"$TESTDIR"/dracut-network-[0-9]*.$(arch).rpm \
+#    ${NULL}
 
     cat >"$rootdir"/test.sh <<EOF
 #!/bin/bash
