--- /usr/share/dracut/modules.d/99base/selinux-loadpolicy.sh~	2009-11-27 09:25:46.000000000 -0500
+++ /usr/share/dracut/modules.d/99base/selinux-loadpolicy.sh	2010-01-22 08:21:25.000000000 -0500
@@ -37,6 +37,8 @@
 	} 2>&1 | vinfo
 
 	if [ $ret -eq 0 -o $ret -eq 2 ]; then
+	    # If machine requires a relabel, force to permissive mode
+	    [ -e "$NEWROOT"/.autorelabel ] && ( echo 0 > "$NEWROOT"/selinux/enforce )
 	    return 0
 	fi
 
