--- dracut-019/modules.d/90kernel-modules/module-setup.sh.orig	2012-06-05 14:48:37.805013809 -0500
+++ dracut-019/modules.d/90kernel-modules/module-setup.sh	2012-06-05 14:48:58.704142946 -0500
@@ -36,7 +36,7 @@
         hostonly='' instmods usb_storage sdhci sdhci-pci
 
         # arm specific modules
-        hostonly='' instmods sdhci_esdhc_imx mmci sdhci_tegra mvsdio omap sdhci_dove ahci_platform pata_imx sata_mv
+        hostonly='' instmods sdhci_esdhc_imx mmci sdhci_tegra mvsdio omap sdhci_dove ahci_platform pata_imx sata_mv omapdrm
 
         # install keyboard support
         hostonly='' instmods atkbd i8042 usbhid hid-apple hid-sunplus hid-cherry hid-logitech hid-logitech-dj hid-microsoft ehci-hcd ohci-hcd uhci-hcd xhci-hcd
